
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://biolink.github.io/biolink-model/">
      
      
      
        <link rel="next" href="using-the-modeling-language/">
      
      
      <link rel="icon" href="assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.15">
    
    
      
        <title>Biolink Model Documentation</title>
      
    
    
      <link rel="stylesheet" href="assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="style.css">
    
    <script>__md_scope=new URL(".",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function e(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],e("js",new Date),e("config","G-J23SCD6JB8"),document.addEventListener("DOMContentLoaded",(function(){document.forms.search&&document.forms.search.query.addEventListener("blur",(function(){this.value&&e("event","search",{search_term:this.value})}));document$.subscribe((function(){var t=document.forms.feedback;if(void 0!==t)for(var a of t.querySelectorAll("[type=submit]"))a.addEventListener("click",(function(a){a.preventDefault();var n=document.location.pathname,d=this.getAttribute("data-md-value");e("event","feedback",{page:n,data:d}),t.firstElementChild.disabled=!0;var r=t.querySelector(".md-feedback__note [data-md-value='"+d+"']");r&&(r.hidden=!1)})),t.hidden=!1})),location$.subscribe((function(t){e("config","G-J23SCD6JB8",{page_path:t.pathname})}))}));var t=document.createElement("script");t.async=!0,t.src="https://www.googletagmanager.com/gtag/js?id=G-J23SCD6JB8",document.getElementById("__analytics").insertAdjacentElement("afterEnd",t)}</script>
  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="black" data-md-color-primary="black" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#biolink-model-a-data-model-for-life-sciences-data" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="." title="Biolink Model Documentation" class="md-header__button md-logo" aria-label="Biolink Model Documentation" data-md-component="logo">
      
  <img src="images/biolink-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Biolink Model Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Overview
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/biolink/biolink-model" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="." title="Biolink Model Documentation" class="md-nav__button md-logo" aria-label="Biolink Model Documentation" data-md-component="logo">
      
  <img src="images/biolink-logo.png" alt="logo">

    </a>
    Biolink Model Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/biolink/biolink-model" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="." class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      Classes (Visualization)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#class-properties" class="md-nav__link">
    <span class="md-ellipsis">
      Class Properties
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#predicates-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      Predicates (Visualization)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qualifiers-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      Qualifiers (Visualization)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#associations" class="md-nav__link">
    <span class="md-ellipsis">
      Associations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#association-properties" class="md-nav__link">
    <span class="md-ellipsis">
      Association Properties
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Subsets">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#citing-biolink-model" class="md-nav__link">
    <span class="md-ellipsis">
      Citing Biolink Model
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="using-the-modeling-language/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Using the LinkML Modeling Language
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="association-examples-with-qualifiers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Associations with Qualifiers
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="reading-a-qualifier-based-statement/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Interpreting a Fully Qualified Edge
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="curating-the-model/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Curating Biolink
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="maintaining-the-model/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Maintaining Biolink
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="working-with-the-model/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Working with Biolink
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://biolink.github.io/information-resource-registry" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Infores Identifier Registry
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_9" >
        
          
          <label class="md-nav__link" for="__nav_9" id="__nav_9_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Visualizations
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_9_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_9">
            <span class="md-nav__icon md-icon"></span>
            Visualizations
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="categories.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Classes (categories)
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="predicates.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Predicates
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="qualifiers.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Qualifiers
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="associations.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Associations
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="aspects.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Aspects
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="faq/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    FAQ
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      Classes (Visualization)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#class-properties" class="md-nav__link">
    <span class="md-ellipsis">
      Class Properties
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#predicates-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      Predicates (Visualization)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#qualifiers-visualization" class="md-nav__link">
    <span class="md-ellipsis">
      Qualifiers (Visualization)
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#associations" class="md-nav__link">
    <span class="md-ellipsis">
      Associations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#association-properties" class="md-nav__link">
    <span class="md-ellipsis">
      Association Properties
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      Enumerations
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      Subsets
    </span>
  </a>
  
    <nav class="md-nav" aria-label="Subsets">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#citing-biolink-model" class="md-nav__link">
    <span class="md-ellipsis">
      Citing Biolink Model
    </span>
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="biolink-model-a-data-model-for-life-sciences-data">Biolink-Model: a data model for life-sciences data.</h1>
<p>The Biolink Model is a high-level, open-source data model designed to standardize types and relationships in biological
knowledge graphs, covering entities like genes, diseases, chemical substances, organisms, genomics, phenotypes, and
more.  Biolink Model can be used to formalize the relationships between data structures in
translational science. It incorporates object-oriented classification and graph-oriented features. The core of the
model is a set of hierarchical, interconnected classes (or categories) and relationships between them (or predicates).
The model provides class and edge attributes and associations that guide how entities should relate to one another.
The goal of the Biolink Model is to provide a consistent framework for representing biological knowledge across various
databases and formats.  For more information, see: <a href="understanding-the-model/">understanding the model</a></p>
<h2 id="classes-visualization">Classes (<a href="categories.html">Visualization</a>)</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AccessibleDnaRegion/">AccessibleDnaRegion</a></td>
<td>A region (or regions) of a chromatinized genome that has been measured to be more accessible to an enzyme such as DNase-I or Tn5 Transpose</td>
</tr>
<tr>
<td><a href="Activity/">Activity</a></td>
<td>An activity is something that occurs over a period of time and acts upon or with entities; it may include consuming, processing, transforming, modifying, relocating, using, or generating entities.</td>
</tr>
<tr>
<td><a href="AdministrativeEntity/">AdministrativeEntity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Agent/">Agent</a></td>
<td>person, group, organization or project that provides a piece of information (i.e. a knowledge association)</td>
</tr>
<tr>
<td><a href="AnatomicalEntity/">AnatomicalEntity</a></td>
<td>A subcellular location, cell type or gross anatomical part</td>
</tr>
<tr>
<td><a href="Annotation/">Annotation</a></td>
<td>Biolink Model root class for entity annotations.</td>
</tr>
<tr>
<td><a href="Article/">Article</a></td>
<td>a piece of writing on a particular topic presented as a stand-alone section of a larger publication</td>
</tr>
<tr>
<td><a href="Attribute/">Attribute</a></td>
<td>A property or characteristic of an entity. For example, an apple may have properties such as color, shape, age, crispiness. An environmental sample may have attributes such as depth, lat, long, material.</td>
</tr>
<tr>
<td><a href="Bacterium/">Bacterium</a></td>
<td>A member of a group of unicellular microorganisms lacking a nuclear membrane, that reproduce by binary fission and are often motile.</td>
</tr>
<tr>
<td><a href="Behavior/">Behavior</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="BehavioralExposure/">BehavioralExposure</a></td>
<td>A behavioral exposure is a factor relating to behavior impacting an individual.</td>
</tr>
<tr>
<td><a href="BehavioralFeature/">BehavioralFeature</a></td>
<td>A phenotypic feature which is behavioral in nature.</td>
</tr>
<tr>
<td><a href="BehavioralOutcome/">BehavioralOutcome</a></td>
<td>An outcome resulting from an exposure event which is the manifestation of human behavior.</td>
</tr>
<tr>
<td><a href="BiologicalEntity/">BiologicalEntity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="BiologicalProcess/">BiologicalProcess</a></td>
<td>One or more causally connected executions of molecular functions</td>
</tr>
<tr>
<td><a href="BiologicalProcessOrActivity/">BiologicalProcessOrActivity</a></td>
<td>Either an individual molecular activity, or a collection of causally connected molecular activities in a biological system.</td>
</tr>
<tr>
<td><a href="BiologicalSex/">BiologicalSex</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="BioticExposure/">BioticExposure</a></td>
<td>An external biotic exposure is an intake of (sometimes pathological) biological organisms (including viruses).</td>
</tr>
<tr>
<td><a href="Book/">Book</a></td>
<td>This class may rarely be instantiated except if use cases of a given knowledge graph support its utility.</td>
</tr>
<tr>
<td><a href="BookChapter/">BookChapter</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Case/">Case</a></td>
<td>An individual (human) organism that has a patient role in some clinical context.</td>
</tr>
<tr>
<td><a href="Cell/">Cell</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="CellLine/">CellLine</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="CellularComponent/">CellularComponent</a></td>
<td>A location in or around a cell</td>
</tr>
<tr>
<td><a href="CellularOrganism/">CellularOrganism</a></td>
<td></td>
</tr>
<tr>
<td><a href="ChemicalEntity/">ChemicalEntity</a></td>
<td>A chemical entity is a physical entity that pertains to chemistry or biochemistry.</td>
</tr>
<tr>
<td><a href="ChemicalExposure/">ChemicalExposure</a></td>
<td>A chemical exposure is an intake of a particular chemical entity.</td>
</tr>
<tr>
<td><a href="ChemicalMixture/">ChemicalMixture</a></td>
<td>A chemical mixture is a chemical entity composed of two or more molecular entities.</td>
</tr>
<tr>
<td><a href="ChemicalRole/">ChemicalRole</a></td>
<td>A role played by the molecular entity or part thereof within a chemical context.</td>
</tr>
<tr>
<td><a href="ChiSquaredAnalysisResult/">ChiSquaredAnalysisResult</a></td>
<td>A result of a chi squared analysis.</td>
</tr>
<tr>
<td><a href="ClinicalAttribute/">ClinicalAttribute</a></td>
<td>Attributes relating to a clinical manifestation</td>
</tr>
<tr>
<td><a href="ClinicalCourse/">ClinicalCourse</a></td>
<td>The course a disease typically takes from its onset, progression in time, and eventual resolution or death of the affected individual</td>
</tr>
<tr>
<td><a href="ClinicalEntity/">ClinicalEntity</a></td>
<td>Any entity or process that exists in the clinical domain and outside the biological realm. Diseases are placed under biological entities</td>
</tr>
<tr>
<td><a href="ClinicalFinding/">ClinicalFinding</a></td>
<td>this category is currently considered broad enough to tag clinical lab measurements and other biological attributes taken as 'clinical traits' with some statistical score, for example, a p value in genetic associations.</td>
</tr>
<tr>
<td><a href="ClinicalIntervention/">ClinicalIntervention</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ClinicalMeasurement/">ClinicalMeasurement</a></td>
<td>A clinical measurement is a special kind of attribute which results from a laboratory observation from a subject individual or sample. Measurements can be connected to their subject by the 'has attribute' slot.</td>
</tr>
<tr>
<td><a href="ClinicalModifier/">ClinicalModifier</a></td>
<td>Used to characterize and specify the phenotypic abnormalities defined in the phenotypic abnormality sub-ontology, with respect to severity, laterality, and other aspects</td>
</tr>
<tr>
<td><a href="ClinicalTrial/">ClinicalTrial</a></td>
<td>A clinical trial is a research study that prospectively assigns human participants or groups of humans to one or more health-related interventions to evaluate the effects on health outcomes.</td>
</tr>
<tr>
<td><a href="CodingSequence/">CodingSequence</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Cohort/">Cohort</a></td>
<td>A group of people banded together or treated as a group who share common characteristics. A cohort 'study' is a particular form of longitudinal study that samples a cohort, performing a cross-section at intervals through time.</td>
</tr>
<tr>
<td><a href="CommonDataElement/">CommonDataElement</a></td>
<td>A Common Data Element (CDE) is a standardized, precisely defined question, paired with a set of allowable responses, used systematically across different sites, studies, or clinical trials to ensure consistent data collection. Multiple CDEs (from one or more Collections) can be curated into Forms. (https://cde.nlm.nih.gov/home)</td>
</tr>
<tr>
<td><a href="ComplexChemicalExposure/">ComplexChemicalExposure</a></td>
<td>A complex chemical exposure is an intake of a chemical mixture (e.g. gasoline), other than a drug.</td>
</tr>
<tr>
<td><a href="ComplexMolecularMixture/">ComplexMolecularMixture</a></td>
<td>A complex molecular mixture is a chemical mixture composed of two or more molecular entities with unknown concentration and stoichiometry.</td>
</tr>
<tr>
<td><a href="ConceptCountAnalysisResult/">ConceptCountAnalysisResult</a></td>
<td>A result of a concept count analysis.</td>
</tr>
<tr>
<td><a href="ConfidenceLevel/">ConfidenceLevel</a></td>
<td>Level of confidence in a statement</td>
</tr>
<tr>
<td><a href="Dataset/">Dataset</a></td>
<td>an item that refers to a collection of data from a data source.</td>
</tr>
<tr>
<td><a href="DatasetDistribution/">DatasetDistribution</a></td>
<td>an item that holds distribution level information about a dataset.</td>
</tr>
<tr>
<td><a href="DatasetSummary/">DatasetSummary</a></td>
<td>an item that holds summary level information about a dataset.</td>
</tr>
<tr>
<td><a href="DatasetVersion/">DatasetVersion</a></td>
<td>an item that holds version level information about a dataset.</td>
</tr>
<tr>
<td><a href="Device/">Device</a></td>
<td>A thing made or adapted for a particular purpose, especially a piece of mechanical or electronic equipment</td>
</tr>
<tr>
<td><a href="DiagnosticAid/">DiagnosticAid</a></td>
<td>A device or substance used to help diagnose disease or injury</td>
</tr>
<tr>
<td><a href="Disease/">Disease</a></td>
<td>A disorder of structure or function, especially one that produces specific signs, phenotypes or symptoms or that affects a specific location and is not simply a direct result of physical injury.  A disposition to undergo pathological processes that exists in an organism because of one or more disorders in that organism.</td>
</tr>
<tr>
<td><a href="DiseaseOrPhenotypicFeature/">DiseaseOrPhenotypicFeature</a></td>
<td>Either one of a disease or an individual phenotypic feature. Some knowledge resources such as Monarch treat these as distinct, others such as MESH conflate.  Please see definitions of phenotypic feature and disease in this model for their independent descriptions.  This class is helpful to enforce domains and ranges that may involve either a disease or a phenotypic feature.</td>
</tr>
<tr>
<td><a href="DiseaseOrPhenotypicFeatureExposure/">DiseaseOrPhenotypicFeatureExposure</a></td>
<td>A disease or phenotypic feature state, when viewed as an exposure, represents an precondition, leading to or influencing an outcome, e.g. HIV predisposing an individual to infections; a relative deficiency of skin pigmentation predisposing an individual to skin cancer.</td>
</tr>
<tr>
<td><a href="DiseaseOrPhenotypicFeatureOutcome/">DiseaseOrPhenotypicFeatureOutcome</a></td>
<td>Physiological outcomes resulting from an exposure event which is the manifestation of a disease or other characteristic phenotype.</td>
</tr>
<tr>
<td><a href="Drug/">Drug</a></td>
<td>A substance intended for use in the diagnosis, cure, mitigation, treatment, or prevention of disease</td>
</tr>
<tr>
<td><a href="DrugExposure/">DrugExposure</a></td>
<td>A drug exposure is an intake of a particular drug.</td>
</tr>
<tr>
<td><a href="DrugLabel/">DrugLabel</a></td>
<td>a document accompanying a drug or its container that provides written, printed or graphic information about the drug, including drug contents, specific instructions or warnings for administration, storage and disposal instructions, etc.</td>
</tr>
<tr>
<td><a href="DrugToGeneInteractionExposure/">DrugToGeneInteractionExposure</a></td>
<td>drug to gene interaction exposure is a drug exposure is where the interactions of the drug with specific genes are known to constitute an 'exposure' to the organism, leading to or influencing an outcome.</td>
</tr>
<tr>
<td><a href="Edge/">Edge</a></td>
<td>An edge in a KGX graph, will be superclass for association</td>
</tr>
<tr>
<td><a href="Entity/">Entity</a></td>
<td>Root Biolink Model class for all things and informational relationships, real or imagined.</td>
</tr>
<tr>
<td><a href="EnvironmentalExposure/">EnvironmentalExposure</a></td>
<td>A environmental exposure is a factor relating to abiotic processes in the environment including sunlight (UV-B), atmospheric (heat, cold, general pollution) and water-born contaminants.</td>
</tr>
<tr>
<td><a href="EnvironmentalFeature/">EnvironmentalFeature</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="EnvironmentalFoodContaminant/">EnvironmentalFoodContaminant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="EnvironmentalProcess/">EnvironmentalProcess</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="EpidemiologicalOutcome/">EpidemiologicalOutcome</a></td>
<td>An epidemiological outcome, such as societal disease burden, resulting from an exposure event.</td>
</tr>
<tr>
<td><a href="Event/">Event</a></td>
<td>Something that happens at a given place and time.</td>
</tr>
<tr>
<td><a href="EvidenceType/">EvidenceType</a></td>
<td>Class of evidence that supports an association</td>
</tr>
<tr>
<td><a href="Exon/">Exon</a></td>
<td>A region of the transcript sequence within a gene which is not removed from the primary RNA transcript by RNA splicing.</td>
</tr>
<tr>
<td><a href="Food/">Food</a></td>
<td>A substance consumed by a living organism as a source of nutrition</td>
</tr>
<tr>
<td><a href="FoodAdditive/">FoodAdditive</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Fungus/">Fungus</a></td>
<td>A kingdom of eukaryotic, heterotrophic organisms that live as saprobes or parasites, including mushrooms, yeasts, smuts, molds, etc. They reproduce either sexually or asexually, and have life cycles that range from simple to complex. Filamentous fungi refer to those that grow as multicellular colonies (mushrooms and molds).</td>
</tr>
<tr>
<td><a href="Gene/">Gene</a></td>
<td>A region (or regions) that includes all of the sequence elements necessary to encode a functional transcript. A gene locus may include regulatory regions, transcribed regions and/or other functional sequence regions.</td>
</tr>
<tr>
<td><a href="GeneFamily/">GeneFamily</a></td>
<td>any grouping of multiple genes or gene products related by common descent</td>
</tr>
<tr>
<td><a href="GeneticInheritance/">GeneticInheritance</a></td>
<td>The pattern or 'mode' in which a particular genetic trait or disorder is passed from one generation to the next, e.g. autosomal dominant, autosomal recessive, etc.</td>
</tr>
<tr>
<td><a href="Genome/">Genome</a></td>
<td>A genome is the sum of genetic material within a cell or virion.</td>
</tr>
<tr>
<td><a href="GenomicBackgroundExposure/">GenomicBackgroundExposure</a></td>
<td>A genomic background exposure is where an individual's specific genomic background of genes, sequence variants or other pre-existing genomic conditions constitute a kind of 'exposure' to the organism, leading to or influencing an outcome.</td>
</tr>
<tr>
<td><a href="Genotype/">Genotype</a></td>
<td>An information content entity that describes a genome by specifying the total variation in genomic sequence and/or gene expression, relative to some established background</td>
</tr>
<tr>
<td><a href="GenotypicSex/">GenotypicSex</a></td>
<td>An attribute corresponding to the genotypic sex of the individual, based upon genotypic composition of sex chromosomes.</td>
</tr>
<tr>
<td><a href="GeographicExposure/">GeographicExposure</a></td>
<td>A geographic exposure is a factor relating to geographic proximity to some impactful entity.</td>
</tr>
<tr>
<td><a href="GeographicLocation/">GeographicLocation</a></td>
<td>a location that can be described in lat/long coordinates</td>
</tr>
<tr>
<td><a href="GeographicLocationAtTime/">GeographicLocationAtTime</a></td>
<td>a location that can be described in lat/long coordinates, for a particular time</td>
</tr>
<tr>
<td><a href="GrossAnatomicalStructure/">GrossAnatomicalStructure</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Haplotype/">Haplotype</a></td>
<td>A set of zero or more Alleles on a single instance of a Sequence[VMC]</td>
</tr>
<tr>
<td><a href="Hospitalization/">Hospitalization</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="HospitalizationOutcome/">HospitalizationOutcome</a></td>
<td>An outcome resulting from an exposure event which is the increased manifestation of acute (e.g. emergency room visit) or chronic (inpatient) hospitalization.</td>
</tr>
<tr>
<td><a href="Human/">Human</a></td>
<td>A member of the the species Homo sapiens.</td>
</tr>
<tr>
<td><a href="IceesStudyResult/">IceesStudyResult</a></td>
<td>A study result that represents a result, from a supporting Study, which is specifically associated with an Integrated Clinical and Environmental Exposures Service (ICEES) knowledge assertion.</td>
</tr>
<tr>
<td><a href="IndividualOrganism/">IndividualOrganism</a></td>
<td>An instance of an organism. For example, Richard Nixon, Charles Darwin, my pet cat. Example ID: ORCID:0000-0002-5355-2576</td>
</tr>
<tr>
<td><a href="InformationContentEntity/">InformationContentEntity</a></td>
<td>a piece of information that typically describes some topic of discourse or is used as support.</td>
</tr>
<tr>
<td><a href="Invertebrate/">Invertebrate</a></td>
<td>An animal lacking a vertebral column. This group consists of 98% of all animal species.</td>
</tr>
<tr>
<td><a href="JournalArticle/">JournalArticle</a></td>
<td>an article, typically presenting results of research, that is published in an issue of a scientific journal.</td>
</tr>
<tr>
<td><a href="KnowledgeGraph/">KnowledgeGraph</a></td>
<td>A knowledge graph is a structured representation of knowledge in the form of a graph, where nodes represent entities or concepts, and edges represent relationships between them. Knowledge graphs are used to organize and connect information from various sources, enabling better understanding, analysis, and reasoning about complex domains.</td>
</tr>
<tr>
<td><a href="KnowledgeGraph/">KnowledgeGraph</a></td>
<td>A knowledge graph represented in KGX format</td>
</tr>
<tr>
<td><a href="LifeStage/">LifeStage</a></td>
<td>A stage of development or growth of an organism, including post-natal adult stages</td>
</tr>
<tr>
<td><a href="LogOddsAnalysisResult/">LogOddsAnalysisResult</a></td>
<td>A result of a log odds ratio analysis.</td>
</tr>
<tr>
<td><a href="MacromolecularComplex/">MacromolecularComplex</a></td>
<td>A stable assembly of two or more macromolecules, i.e. proteins, nucleic acids, carbohydrates or lipids, in which at least one component is a protein and the constituent parts function together.</td>
</tr>
<tr>
<td><a href="Mammal/">Mammal</a></td>
<td>A member of the class Mammalia, a clade of endothermic amniotes distinguished from reptiles and birds by the possession of hair, three middle ear bones, mammary glands, and a neocortex</td>
</tr>
<tr>
<td><a href="MappingCollection/">MappingCollection</a></td>
<td>A collection of deprecated mappings.</td>
</tr>
<tr>
<td><a href="MaterialSample/">MaterialSample</a></td>
<td>A sample is a limited quantity of something (e.g. an individual or set of individuals from a population, or a portion of a substance) to be used for testing, analysis, inspection, investigation, demonstration, or trial use. [SIO]</td>
</tr>
<tr>
<td><a href="MicroRNA/">MicroRNA</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="MolecularActivity/">MolecularActivity</a></td>
<td>An execution of a molecular function carried out by a gene product or macromolecular complex.</td>
</tr>
<tr>
<td><a href="MolecularEntity/">MolecularEntity</a></td>
<td>A molecular entity is a chemical entity composed of individual or covalently bonded atoms.</td>
</tr>
<tr>
<td><a href="MolecularMixture/">MolecularMixture</a></td>
<td>A molecular mixture is a chemical mixture composed of two or more molecular entities with known concentration and stoichiometry.</td>
</tr>
<tr>
<td><a href="MortalityOutcome/">MortalityOutcome</a></td>
<td>An outcome of death from resulting from an exposure event.</td>
</tr>
<tr>
<td><a href="NamedThing/">NamedThing</a></td>
<td>a databased entity or concept/class</td>
</tr>
<tr>
<td><a href="Node/">Node</a></td>
<td>A node in a KGX graph, will be superclass for named thing</td>
</tr>
<tr>
<td><a href="NoncodingRNAProduct/">NoncodingRNAProduct</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="NucleicAcidEntity/">NucleicAcidEntity</a></td>
<td>A nucleic acid entity is a molecular entity characterized by availability in gene databases of nucleotide-based sequence representations of its precise sequence; for convenience of representation, partial sequences of various kinds are included.</td>
</tr>
<tr>
<td><a href="NucleicAcidSequenceMotif/">NucleicAcidSequenceMotif</a></td>
<td>A linear nucleotide sequence pattern that is widespread and has, or is conjectured to have, a biological significance. e.g. the TATA box promoter motif, transcription factor binding consensus sequences.</td>
</tr>
<tr>
<td><a href="NucleosomeModification/">NucleosomeModification</a></td>
<td>A chemical modification of a histone protein within a nucleosome octomer or a substitution of a histone with a variant histone isoform. e.g. Histone 4 Lysine 20 methylation (H4K20me), histone variant H2AZ substituting H2A.</td>
</tr>
<tr>
<td><a href="ObservedExpectedFrequencyAnalysisResult/">ObservedExpectedFrequencyAnalysisResult</a></td>
<td>A result of a observed expected frequency analysis.</td>
</tr>
<tr>
<td><a href="Onset/">Onset</a></td>
<td>The age group in which (disease) symptom manifestations appear.</td>
</tr>
<tr>
<td><a href="OrganismAttribute/">OrganismAttribute</a></td>
<td>describes a characteristic of an organismal entity.</td>
</tr>
<tr>
<td><a href="OrganismTaxon/">OrganismTaxon</a></td>
<td>A classification of a set of organisms. Example instances: NCBITaxon:9606 (Homo sapiens), NCBITaxon:2 (Bacteria). Can also be used to represent strains or subspecies.</td>
</tr>
<tr>
<td><a href="OrganismalEntity/">OrganismalEntity</a></td>
<td>A named entity that is either a part of an organism, a whole organism, population or clade of organisms, excluding chemical entities</td>
</tr>
<tr>
<td><a href="Patent/">Patent</a></td>
<td>a legal document granted by a patent issuing authority which confers upon the patenter the sole right to make, use and sell an invention for a set period of time.</td>
</tr>
<tr>
<td><a href="PathologicalAnatomicalExposure/">PathologicalAnatomicalExposure</a></td>
<td>An abnormal anatomical structure, when viewed as an exposure, representing an precondition, leading to or influencing an outcome, e.g. thrombosis leading to an ischemic disease outcome.</td>
</tr>
<tr>
<td><a href="PathologicalAnatomicalOutcome/">PathologicalAnatomicalOutcome</a></td>
<td>An outcome resulting from an exposure event which is the manifestation of an abnormal anatomical structure.</td>
</tr>
<tr>
<td><a href="PathologicalAnatomicalStructure/">PathologicalAnatomicalStructure</a></td>
<td>An anatomical structure with the potential of have an abnormal or deleterious effect at the subcellular, cellular, multicellular, or organismal level.</td>
</tr>
<tr>
<td><a href="PathologicalProcess/">PathologicalProcess</a></td>
<td>A biologic function or a process having an abnormal or deleterious effect at the subcellular, cellular, multicellular, or organismal level.</td>
</tr>
<tr>
<td><a href="PathologicalProcessExposure/">PathologicalProcessExposure</a></td>
<td>A pathological process, when viewed as an exposure, representing a precondition, leading to or influencing an outcome, e.g. autoimmunity leading to disease.</td>
</tr>
<tr>
<td><a href="PathologicalProcessOutcome/">PathologicalProcessOutcome</a></td>
<td>An outcome resulting from an exposure event which is the manifestation of a pathological process.</td>
</tr>
<tr>
<td><a href="Pathway/">Pathway</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="Phenomenon/">Phenomenon</a></td>
<td>a fact or situation that is observed to exist or happen, especially one whose cause or explanation is in question</td>
</tr>
<tr>
<td><a href="PhenotypicFeature/">PhenotypicFeature</a></td>
<td>A combination of entity and quality that makes up a phenotyping statement. An observable characteristic of an individual resulting from the interaction of its genotype with its molecular and physical environment.</td>
</tr>
<tr>
<td><a href="PhenotypicQuality/">PhenotypicQuality</a></td>
<td>A property of a phenotype</td>
</tr>
<tr>
<td><a href="PhenotypicSex/">PhenotypicSex</a></td>
<td>An attribute corresponding to the phenotypic sex of the individual, based upon the reproductive organs present.</td>
</tr>
<tr>
<td><a href="PhysicalEntity/">PhysicalEntity</a></td>
<td>An entity that has material reality (a.k.a. physical essence).</td>
</tr>
<tr>
<td><a href="PhysiologicalProcess/">PhysiologicalProcess</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="PlanetaryEntity/">PlanetaryEntity</a></td>
<td>Any entity or process that exists at the level of the whole planet</td>
</tr>
<tr>
<td><a href="Plant/">Plant</a></td>
<td></td>
</tr>
<tr>
<td><a href="Polypeptide/">Polypeptide</a></td>
<td>A polypeptide is a molecular entity characterized by availability in protein databases of amino-acid-based sequence representations of its precise primary structure; for convenience of representation, partial sequences of various kinds are included, even if they do not represent a physical molecule.</td>
</tr>
<tr>
<td><a href="PopulationOfIndividualOrganisms/">PopulationOfIndividualOrganisms</a></td>
<td>A collection of individuals from the same taxonomic class distinguished by one or more characteristics.  Characteristics can include, but are not limited to, shared geographic location, genetics, phenotypes.</td>
</tr>
<tr>
<td><a href="PosttranslationalModification/">PosttranslationalModification</a></td>
<td>A chemical modification of a polypeptide or protein that occurs after translation. e.g. polypeptide cleavage to form separate proteins, methylation or acetylation of histone tail amino acids, protein ubiquitination.</td>
</tr>
<tr>
<td><a href="PredicateMapping/">PredicateMapping</a></td>
<td>A deprecated predicate mapping object contains the deprecated predicate and an example of the rewiring that should be done to use a qualified statement in its place.</td>
</tr>
<tr>
<td><a href="PreprintPublication/">PreprintPublication</a></td>
<td>a document reresenting an early version of an author's original scholarly work, such as a research paper or a review, prior to formal peer review and publication in a peer-reviewed scholarly or scientific journal.</td>
</tr>
<tr>
<td><a href="Procedure/">Procedure</a></td>
<td>A series of actions conducted in a certain order or manner</td>
</tr>
<tr>
<td><a href="ProcessedMaterial/">ProcessedMaterial</a></td>
<td>A chemical entity (often a mixture) processed for consumption for nutritional, medical or technical use. Is a material entity that is created or changed during material processing.</td>
</tr>
<tr>
<td><a href="Protein/">Protein</a></td>
<td>A gene product that is composed of a chain of amino acid sequences and is produced by ribosome-mediated translation of mRNA</td>
</tr>
<tr>
<td><a href="ProteinDomain/">ProteinDomain</a></td>
<td>A conserved part of protein sequence and (tertiary) structure that can evolve, function, and exist independently of the rest of the protein chain. Protein domains maintain their structure and function independently of the proteins in which they are found. e.g. an SH3 domain.</td>
</tr>
<tr>
<td><a href="ProteinFamily/">ProteinFamily</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ProteinIsoform/">ProteinIsoform</a></td>
<td>Represents a protein that is a specific isoform of the canonical or reference protein. See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4114032/</td>
</tr>
<tr>
<td><a href="Publication/">Publication</a></td>
<td>Any ‘published’ piece of information. Publications are considered broadly to include any document or document part made available in print or on the web - which may include scientific journal issues, individual articles, and books - as well as things like pre-prints, white papers, patents, drug labels, web pages, protocol documents,  and even a part of a publication if of significant knowledge scope (e.g. a figure, figure legend, or section highlighted by NLP).</td>
</tr>
<tr>
<td><a href="QuantityValue/">QuantityValue</a></td>
<td>A value of an attribute that is quantitative and measurable, expressed as a combination of a unit and a numeric value</td>
</tr>
<tr>
<td><a href="ReagentTargetedGene/">ReagentTargetedGene</a></td>
<td>A gene altered in its expression level in the context of some experiment as a result of being targeted by gene-knockdown reagent(s) such as a morpholino or RNAi.</td>
</tr>
<tr>
<td><a href="RegulatoryRegion/">RegulatoryRegion</a></td>
<td>A region (or regions) of the genome that contains known or putative regulatory elements that act in cis- or trans- to affect the transcription of gene</td>
</tr>
<tr>
<td><a href="RelationshipType/">RelationshipType</a></td>
<td>An OWL property used as an edge label</td>
</tr>
<tr>
<td><a href="RelativeFrequencyAnalysisResult/">RelativeFrequencyAnalysisResult</a></td>
<td>A result of a relative frequency analysis.</td>
</tr>
<tr>
<td><a href="RetrievalSource/">RetrievalSource</a></td>
<td>Provides information about how a particular InformationResource served as a source from which knowledge expressed in an Edge, or data used to generate this knowledge, was retrieved.</td>
</tr>
<tr>
<td><a href="RNAProduct/">RNAProduct</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="RNAProductIsoform/">RNAProductIsoform</a></td>
<td>Represents a protein that is a specific isoform of the canonical or reference RNA</td>
</tr>
<tr>
<td><a href="SequenceVariant/">SequenceVariant</a></td>
<td>A sequence_variant is a non exact copy of a sequence_feature or genome exhibiting one or more sequence_alteration.</td>
</tr>
<tr>
<td><a href="Serial/">Serial</a></td>
<td>This class may rarely be instantiated except if use cases of a given knowledge graph support its utility.</td>
</tr>
<tr>
<td><a href="SeverityValue/">SeverityValue</a></td>
<td>describes the severity of a phenotypic feature or disease</td>
</tr>
<tr>
<td><a href="SiRNA/">SiRNA</a></td>
<td>A small RNA molecule that is the product of a longer exogenous or endogenous dsRNA, which is either a bimolecular duplex or very long hairpin, processed (via the Dicer pathway) such that numerous siRNAs accumulate from both strands of the dsRNA. SRNAs trigger the cleavage of their target molecules.</td>
</tr>
<tr>
<td><a href="SmallMolecule/">SmallMolecule</a></td>
<td>A small molecule entity is a molecular entity characterized by availability in small-molecule databases of SMILES, InChI, IUPAC, or other unambiguous representation of its precise chemical structure; for convenience of representation, any valid chemical representation is included, even if it is not strictly molecular (e.g., sodium ion).</td>
</tr>
<tr>
<td><a href="Snv/">Snv</a></td>
<td>SNVs are single nucleotide positions in genomic DNA at which different sequence alternatives exist</td>
</tr>
<tr>
<td><a href="SocioeconomicAttribute/">SocioeconomicAttribute</a></td>
<td>Attributes relating to a socioeconomic manifestation</td>
</tr>
<tr>
<td><a href="SocioeconomicExposure/">SocioeconomicExposure</a></td>
<td>A socioeconomic exposure is a factor relating to social and financial status of an affected individual (e.g. poverty).</td>
</tr>
<tr>
<td><a href="SocioeconomicOutcome/">SocioeconomicOutcome</a></td>
<td>An general social or economic outcome, such as healthcare costs, utilization, etc., resulting from an exposure event</td>
</tr>
<tr>
<td><a href="Study/">Study</a></td>
<td>a detailed investigation and/or analysis</td>
</tr>
<tr>
<td><a href="StudyPopulation/">StudyPopulation</a></td>
<td>A group of people banded together or treated as a group as participants in a research study.</td>
</tr>
<tr>
<td><a href="StudyResult/">StudyResult</a></td>
<td>A collection of data items from a study that are about a particular study subject or experimental unit (the 'focus' of the Result) - optionally with context/provenance metadata that may be relevant to the interpretation of this data as evidence.</td>
</tr>
<tr>
<td><a href="StudyVariable/">StudyVariable</a></td>
<td>a variable that is used as a measure in the investigation of a study</td>
</tr>
<tr>
<td><a href="TaxonomicRank/">TaxonomicRank</a></td>
<td>A descriptor for the rank within a taxonomic classification. Example instance: TAXRANK:0000017 (kingdom)</td>
</tr>
<tr>
<td><a href="TextMiningStudyResult/">TextMiningStudyResult</a></td>
<td>A study result that represents information extracted from text using natural language processing techniques. This includes the extracted text, location offsets within the source document, confidence scores, and other metadata related to the text mining process.</td>
</tr>
<tr>
<td><a href="Transcript/">Transcript</a></td>
<td>An RNA synthesized on a DNA or RNA template by an RNA polymerase.</td>
</tr>
<tr>
<td><a href="TranscriptionFactorBindingSite/">TranscriptionFactorBindingSite</a></td>
<td>A region (or regions) of the genome that contains a region of DNA known or predicted to bind a protein that modulates gene transcription</td>
</tr>
<tr>
<td><a href="Treatment/">Treatment</a></td>
<td>A treatment is targeted at a disease or phenotype and may involve multiple drug 'exposures', medical devices and/or procedures</td>
</tr>
<tr>
<td><a href="Vertebrate/">Vertebrate</a></td>
<td>A sub-phylum of animals consisting of those having a bony or cartilaginous vertebral column.</td>
</tr>
<tr>
<td><a href="Virus/">Virus</a></td>
<td>A virus is a microorganism that replicates itself as a microRNA and infects the host cell.</td>
</tr>
<tr>
<td><a href="WebPage/">WebPage</a></td>
<td>a document that is published according to World Wide Web standards, which may incorporate text, graphics, sound, and/or other features.</td>
</tr>
<tr>
<td><a href="Zygosity/">Zygosity</a></td>
<td>None</td>
</tr>
</tbody>
</table>
<h2 id="class-properties">Class Properties</h2>
<table>
<thead>
<tr>
<th>Property</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="address/">address</a></td>
<td>the particulars of the place where someone or an organization is situated.  For now, this slot is a simple text "blob" containing all relevant details of the given location for fitness of purpose. For the moment, this "address" can include other contact details such as email and phone number(?).</td>
</tr>
<tr>
<td><a href="affiliation/">affiliation</a></td>
<td>a professional relationship between one provider (often a person) within another provider (often an organization). Target provider identity should be specified by a CURIE. Providers may have multiple affiliations.</td>
</tr>
<tr>
<td><a href="aggregate_statistic/">aggregate_statistic</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="animal_model_available_from/">animal_model_available_from</a></td>
<td></td>
</tr>
<tr>
<td><a href="authors/">authors</a></td>
<td>connects an publication to the list of authors who contributed to the publication. This property should be a comma-delimited list of author names. It is recommended that an author's name be formatted as "surname, firstname initial.".   Note that this property is a node annotation expressing the citation list of authorship which might typically otherwise be more completely documented in biolink:PublicationToProviderAssociation defined edges which point to full details about an author and possibly, some qualifiers which clarify the specific status of a given author in the publication.</td>
</tr>
<tr>
<td><a href="available_from/">available_from</a></td>
<td></td>
</tr>
<tr>
<td><a href="broad_synonym/">broad_synonym</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="chapter/">chapter</a></td>
<td>chapter of a book</td>
</tr>
<tr>
<td><a href="chembl_chirality/">chembl_chirality</a></td>
<td>Tern indicating the chirality of the chemical entity.</td>
</tr>
<tr>
<td><a href="chembl_drug_warning/">chembl_drug_warning</a></td>
<td>Text describing warnings for use of chemicals as therapeutics.</td>
</tr>
<tr>
<td><a href="chembl_prodrug/">chembl_prodrug</a></td>
<td>Flag indicating if a drug is a prodrug that is active only after being metabolized by the body.</td>
</tr>
<tr>
<td><a href="clinical_trial_intervention_boxed_warning/">clinical_trial_intervention_boxed_warning</a></td>
<td>A boolean flag indicating whether a clinical trial intervention has a boxed warning. A boxed warning is the strongest warning that the FDA requires on a prescription drug label. This property should be populated from DailyMed.</td>
</tr>
<tr>
<td><a href="clinical_trial_intervention_model/">clinical_trial_intervention_model</a></td>
<td>The intervention model of a clinical trial as determined by clinicaltrials.gov.  The most common values are SINGLE_GROUP, PARALLEL, CROSSOVER, FACTORIAL, and (null).</td>
</tr>
<tr>
<td><a href="clinical_trial_overall_status/">clinical_trial_overall_status</a></td>
<td>The overall status of a clinical trial as determined by clinicaltrials.gov</td>
</tr>
<tr>
<td><a href="clinical_trial_phase/">clinical_trial_phase</a></td>
<td>The phase that a clinical trials study represents</td>
</tr>
<tr>
<td><a href="clinical_trial_primary_purpose/">clinical_trial_primary_purpose</a></td>
<td>The primary purpose of a clinical trial as determined by clinicaltrials.gov.  The most common values are TREATMENT and PREVENTION. Other possible values include BASIC_SCIENCE, SUPPORTIVE_CARE, DIAGNOSTIC, HEALTH_SERVICES_RESEARCH, SCREENING, DEVICE_FEASIBILITY, OTHER, and (null).</td>
</tr>
<tr>
<td><a href="clinical_trial_tested_intervention/">clinical_trial_tested_intervention</a></td>
<td>Records whether the clinical trials are testing the intervention.</td>
</tr>
<tr>
<td><a href="clinical_trial_time_perspective/">clinical_trial_time_perspective</a></td>
<td>The time perspective of a clinical trial as determined by clinicaltrials.gov.  The most common values are PROSPECTIVE, RETROSPECTIVE, CROSS_SECTIONAL, or OTHER (most common)..</td>
</tr>
<tr>
<td><a href="created_with/">created_with</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="creation_date/">creation_date</a></td>
<td>date on which an entity was created. This can be applied to nodes or edges</td>
</tr>
<tr>
<td><a href="dataset_download_url/">dataset_download_url</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="distribution_download_url/">distribution_download_url</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="download_url/">download_url</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="equivalent_identifiers/">equivalent_identifiers</a></td>
<td>A set of identifiers that are considered equivalent to the primary identifier of the entity. This attribute is used to represent a collection of identifiers that are considered equivalent to the primary identifier of an entity. These equivalent identifiers may come from different databases, ontologies, or naming conventions, but they all refer to the same underlying concept or entity. This attribute is particularly useful in data integration and interoperability scenarios, where it is important to recognize and link different representations of the same entity across various sources.</td>
</tr>
<tr>
<td><a href="exact_synonym/">exact_synonym</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="format/">format</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="full_name/">full_name</a></td>
<td>a long-form human readable name for a thing</td>
</tr>
<tr>
<td><a href="has_biological_sequence/">has_biological_sequence</a></td>
<td>connects a genomic feature to its sequence</td>
</tr>
<tr>
<td><a href="has_chemical_formula/">has_chemical_formula</a></td>
<td>description of chemical compound based on element symbols</td>
</tr>
<tr>
<td><a href="has_constituent/">has_constituent</a></td>
<td>one or more molecular entities within a chemical mixture</td>
</tr>
<tr>
<td><a href="has_count/">has_count</a></td>
<td>number of things with a particular property</td>
</tr>
<tr>
<td><a href="has_dataset/">has_dataset</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_device/">has_device</a></td>
<td>connects an entity to one or more (medical) devices</td>
</tr>
<tr>
<td><a href="has_distribution/">has_distribution</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_drug/">has_drug</a></td>
<td>connects an entity to one or more drugs</td>
</tr>
<tr>
<td><a href="has_gene/">has_gene</a></td>
<td>connects an entity associated with one or more genes</td>
</tr>
<tr>
<td><a href="has_gene_or_gene_product/">has_gene_or_gene_product</a></td>
<td>connects an entity with one or more gene or gene products</td>
</tr>
<tr>
<td><a href="has_percentage/">has_percentage</a></td>
<td>equivalent to has quotient multiplied by 100</td>
</tr>
<tr>
<td><a href="has_procedure/">has_procedure</a></td>
<td>connects an entity to one or more (medical) procedures</td>
</tr>
<tr>
<td><a href="has_quotient/">has_quotient</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_receptor/">has_receptor</a></td>
<td>the organism or organism part being exposed</td>
</tr>
<tr>
<td><a href="has_route/">has_route</a></td>
<td>the process that results in the stressor coming into direct contact with the receptor</td>
</tr>
<tr>
<td><a href="has_stressor/">has_stressor</a></td>
<td>the process or entity that the receptor is being exposed to</td>
</tr>
<tr>
<td><a href="has_taxonomic_rank/">has_taxonomic_rank</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_topic/">has_topic</a></td>
<td>Connects a node to a vocabulary term or ontology class that describes some aspect of the entity. In general specific characterization is preferred. See https://github.com/biolink/biolink-model/issues/238</td>
</tr>
<tr>
<td><a href="has_total/">has_total</a></td>
<td>total number of things in a particular reference set</td>
</tr>
<tr>
<td><a href="has_zygosity/">has_zygosity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="hgvs_nomenclature/">hgvs_nomenclature</a></td>
<td>HGVS syntax refers to the specific rules and conventions used by the Human Variant Nomenclature Committee to describe the location and change in DNA, RNA, and protein sequence variants.  This slot is used to capture all the different forms of HGVS nomenclature that may be used to describe a sequence variant, including genomic, transcript, and protein HGVS expressions/nomenclatures and is thus multivalued.</td>
</tr>
<tr>
<td><a href="in_taxon_label/">in_taxon_label</a></td>
<td>The human readable scientific name for the taxon of the entity.</td>
</tr>
<tr>
<td><a href="information_content/">information_content</a></td>
<td>Information content (IC) value for a term, primarily from Automats.</td>
</tr>
<tr>
<td><a href="ingest_date/">ingest_date</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="inheritance/">inheritance</a></td>
<td>Connects genetic inheritance to a disease or phenotypic feature, as a node property.</td>
</tr>
<tr>
<td><a href="is_metabolite/">is_metabolite</a></td>
<td>indicates whether a molecular entity is a metabolite</td>
</tr>
<tr>
<td><a href="is_supplement/">is_supplement</a></td>
<td></td>
</tr>
<tr>
<td><a href="is_toxic/">is_toxic</a></td>
<td></td>
</tr>
<tr>
<td><a href="iso_abbreviation/">iso_abbreviation</a></td>
<td>Standard abbreviation for periodicals in the International Organization for Standardization (ISO) 4 system See https://www.issn.org/services/online-services/access-to-the-ltwa/. If the 'published in' property is set, then the iso abbreviation pertains to the broader publication context (the journal) within which the given publication node is embedded, not the publication itself.</td>
</tr>
<tr>
<td><a href="issue/">issue</a></td>
<td>issue of a newspaper, a scientific journal or magazine for reference purpose</td>
</tr>
<tr>
<td><a href="keywords/">keywords</a></td>
<td>keywords tagging a publication</td>
</tr>
<tr>
<td><a href="latitude/">latitude</a></td>
<td>latitude</td>
</tr>
<tr>
<td><a href="license/">license</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="longitude/">longitude</a></td>
<td>longitude</td>
</tr>
<tr>
<td><a href="max_tolerated_dose/">max_tolerated_dose</a></td>
<td>The highest dose of a drug or treatment that does not cause unacceptable side effects. The maximum tolerated dose is determined in clinical trials by testing increasing doses on different groups of people until the highest dose with acceptable side effects is found. Also called MTD.</td>
</tr>
<tr>
<td><a href="mesh_terms/">mesh_terms</a></td>
<td>mesh terms tagging a publication</td>
</tr>
<tr>
<td><a href="narrow_synonym/">narrow_synonym</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="node_property/">node_property</a></td>
<td>A grouping for any property that holds between a node and a value</td>
</tr>
<tr>
<td><a href="pages/">pages</a></td>
<td>page number of source referenced for statement or publication</td>
</tr>
<tr>
<td><a href="provided_by/">provided_by</a></td>
<td>The value in this node property represents the knowledge provider that created or assembled the node and all of its attributes.  Used internally to represent how a particular node made its way into a knowledge provider or graph.</td>
</tr>
<tr>
<td><a href="published_in/">published_in</a></td>
<td>CURIE identifier of a broader publication context within which the publication may be placed, e.g. a specified book or journal.</td>
</tr>
<tr>
<td><a href="related_synonym/">related_synonym</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="resource_id/">resource_id</a></td>
<td>The CURIE for an Information Resource that served as a source of knowledge expressed in an Edge, or a source of data used to generate this knowledge.</td>
</tr>
<tr>
<td><a href="resource_role/">resource_role</a></td>
<td>The role played by the InformationResource in serving as a source for an Edge. Note that a given Edge should have one and only one 'primary' source, and may have any number of 'aggregator' or 'supporting data' sources.</td>
</tr>
<tr>
<td><a href="retrieved_on/">retrieved_on</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="rights/">rights</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="source_logo/">source_logo</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="source_record_urls/">source_record_urls</a></td>
<td>A URL linking to a specific web page or document provided by the source, that contains a record of the knowledge expressed in the Edge. If the knowledge is contained in more than one web page on an Information Resource's site, urls MAY be provided for each.</td>
</tr>
<tr>
<td><a href="source_web_page/">source_web_page</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="summary/">summary</a></td>
<td>executive  summary of a publication</td>
</tr>
<tr>
<td><a href="symbol/">symbol</a></td>
<td>Symbol for a particular thing</td>
</tr>
<tr>
<td><a href="synonym/">synonym</a></td>
<td>Alternate human-readable names for a thing</td>
</tr>
<tr>
<td><a href="systematic_synonym/">systematic_synonym</a></td>
<td>more commonly used for gene symbols in yeast</td>
</tr>
<tr>
<td><a href="trade_name/">trade_name</a></td>
<td></td>
</tr>
<tr>
<td><a href="update_date/">update_date</a></td>
<td>date on which an entity was updated. This can be applied to nodes or edges</td>
</tr>
<tr>
<td><a href="upstream_resource_ids/">upstream_resource_ids</a></td>
<td>An upstream InformationResource from which the resource being described directly retrieved a record of the knowledge expressed in the Edge, or data used to generate this knowledge. This is an array because there are cases where a merged Edge holds knowledge that was retrieved from multiple sources. e.g. an Edge provided by the ARAGORN ARA can expressing knowledge it retrieved from both the automat-mychem-info and molepro KPs, which both provided it with records of this single fact.</td>
</tr>
<tr>
<td><a href="url/">url</a></td>
<td>This slot holds a string representation of a URL for an external resource about the node it is present on. Unlike an 'xref' that is primarily represented by a CURIE, this slot is intended to hold a full URL that can be used to directly access a resource. When linking to an external resource that cannot be represented by a unique CURIE, this slot should be used.  However, when the intent is to link to the default URI expansion of a CURIE related to the node it is present on, the xref slot should be used instead.</td>
</tr>
<tr>
<td><a href="version/">version</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="version_of/">version_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="volume/">volume</a></td>
<td>volume of a book or music release in a collection/series or a published collection of journal issues in a serial publication</td>
</tr>
</tbody>
</table>
<h2 id="predicates-visualization">Predicates (<a href="predicates.html">Visualization</a>)</h2>
<table>
<thead>
<tr>
<th>Property</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="active_in/">active_in</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="actively_involved_in/">actively_involved_in</a></td>
<td>holds between a continuant and a process or function, where the continuant actively contributes to part or all of the process or function it realizes</td>
</tr>
<tr>
<td><a href="actively_involves/">actively_involves</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="acts_upstream_of/">acts_upstream_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="acts_upstream_of_negative_effect/">acts_upstream_of_negative_effect</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="acts_upstream_of_or_within/">acts_upstream_of_or_within</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="acts_upstream_of_or_within_negative_effect/">acts_upstream_of_or_within_negative_effect</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="acts_upstream_of_or_within_positive_effect/">acts_upstream_of_or_within_positive_effect</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="acts_upstream_of_positive_effect/">acts_upstream_of_positive_effect</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="adverse_event_of/">adverse_event_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="affected_by/">affected_by</a></td>
<td>describes an entity of which the state or quality is affected by another existing entity.</td>
</tr>
<tr>
<td><a href="affects/">affects</a></td>
<td>Describes an entity that has an effect on the state or quality of another existing entity.</td>
</tr>
<tr>
<td><a href="affects_likelihood_of/">affects_likelihood_of</a></td>
<td>Holds between two entities where the presence or application of one alters the chance that the other will come to be.</td>
</tr>
<tr>
<td><a href="affects_sensitivity_to/">affects_sensitivity_to</a></td>
<td>holds between two chemical entities or genes or gene products where the action of one affects the susceptibility/sensitivity of a biological entity or system (e.g. an organism, cell, cellular component, macromolecular machine mixin, biological or pathological process) to the other.</td>
</tr>
<tr>
<td><a href="ameliorates_condition/">ameliorates_condition</a></td>
<td>Holds between an entity and an existing medical condition (disease or phenotypic feature) where the entity is able to ameliorate symptoms, stabilize progression, or cure the condition.</td>
</tr>
<tr>
<td><a href="amount_or_activity_decreased_by/">amount_or_activity_decreased_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="amount_or_activity_increased_by/">amount_or_activity_increased_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="applied_to_treat/">applied_to_treat</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that the  substance, procedure, or activity was actually taken by one or more patients with the intent of treating the condition.</td>
</tr>
<tr>
<td><a href="associated_with/">associated_with</a></td>
<td>Expresses a relationship between two named things where the relationship is typically generated statistically (though not in all cases), and is weaker than its child, 'correlated with', but stronger than its parent, 'related to'. This relationship holds between two concepts represented by variables for which a statistical dependence is demonstrated.  E.g. the statement “Atrial Fibrillation (Afib) is associated with Myocardial Infraction (MI)” asserts that having Afib is not statistically independent from whether a patient will also have MI. Note that in Translator associations, the subject and object concepts may map exactly to the statistical variables, or represent related entities for which the variables serve as proxies in an Association (e.g. diseases, chemical entities or processes).</td>
</tr>
<tr>
<td><a href="associated_with_decreased_likelihood_of/">associated_with_decreased_likelihood_of</a></td>
<td>Expresses a relationship between two named things where the relationship is typically generated statistically and the state or fact of something is less probable.</td>
</tr>
<tr>
<td><a href="associated_with_increased_likelihood_of/">associated_with_increased_likelihood_of</a></td>
<td>Expresses a relationship between two named things where the relationship is typically generated statistically and the state or fact of something is more probable.</td>
</tr>
<tr>
<td><a href="associated_with_likelihood_of/">associated_with_likelihood_of</a></td>
<td>A a relationship that holds between two concepts represented by variables for which a statistical dependence is demonstrated, wherein the state or value of one variable predicts the future state or value of the other.  E.g. the statement “An Atrial Fibrillation (Afib) diagnosis is associated with likelihood of a Myocardial Infraction (MI) diagnosis” asserts that the state of having Afib is associated with an increased or decreased likelihood that a patient will later exhibit MI.</td>
</tr>
<tr>
<td><a href="associated_with_resistance_to/">associated_with_resistance_to</a></td>
<td>A relation that holds between a named thing and a chemical that specifies that the change in the named thing is found to be associated with the degree of resistance to treatment by the chemical.</td>
</tr>
<tr>
<td><a href="associated_with_response_to/">associated_with_response_to</a></td>
<td>A statistical association used to indicate that the object of a statement using this predicate induces a response of some kind in the subject entity.  Intentionally broad in definition, this predicate should be used with qualifiers to narrow the type of response (E.g. whether the response is therapeutic, phenotypic, detrimental, resistant, etc. is captured in context, direction, and aspect qualifiers).</td>
</tr>
<tr>
<td><a href="associated_with_sensitivity_to/">associated_with_sensitivity_to</a></td>
<td>A relation that holds between a named thing and a chemical that specifies that the change in the named thing is found to be associated with the degree of sensitivity to treatment by the chemical.</td>
</tr>
<tr>
<td><a href="author/">author</a></td>
<td>an instance of one (co-)creator primarily responsible for a written work</td>
</tr>
<tr>
<td><a href="beneficial_in_models_for/">beneficial_in_models_for</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that the substance, procedure, or activity has been shown to be effective in alleviating, preventing, or delaying symptoms/ phenotypes associated with a disease, in a model system for that disease (e.g. a mouse, fly, cell line, etc).</td>
</tr>
<tr>
<td><a href="binds/">binds</a></td>
<td>A causal mechanism mediated by the direct contact between effector and target chemical or biomolecular entity, which form a stable physical interaction.</td>
</tr>
<tr>
<td><a href="biomarker_for/">biomarker_for</a></td>
<td>holds between a measurable chemical entity and a disease or phenotypic feature, where the entity is used as an indicator of the presence or state of the disease or feature.</td>
</tr>
<tr>
<td><a href="broad_match/">broad_match</a></td>
<td>a list of terms from different schemas or terminology systems that have a broader, more general meaning. Broader terms are typically shown as parents in a hierarchy or tree.</td>
</tr>
<tr>
<td><a href="can_be_carried_out_by/">can_be_carried_out_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="capable_of/">capable_of</a></td>
<td>holds between a physical entity and process or function, where the continuant alone has the ability to carry out the process or function.</td>
</tr>
<tr>
<td><a href="catalyzes/">catalyzes</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="caused_by/">caused_by</a></td>
<td>holds between two entities where the occurrence, existence, or activity of one is caused by the occurrence or generation of the other</td>
</tr>
<tr>
<td><a href="causes/">causes</a></td>
<td>holds between two entities where the occurrence, existence, or activity of one causes the occurrence or generation of the other</td>
</tr>
<tr>
<td><a href="chemically_similar_to/">chemically_similar_to</a></td>
<td>holds between one small molecule entity and another that it approximates for purposes of scientific study, in virtue of its exhibiting similar features of the studied entity.</td>
</tr>
<tr>
<td><a href="close_match/">close_match</a></td>
<td>a list of terms from different schemas or terminology systems that have a semantically similar but not strictly equivalent, broader, or narrower meaning. Such terms often describe the same general concept from different ontological perspectives (e.g. drug as a type of chemical entity versus drug as a type of role borne by a chemical entity).</td>
</tr>
<tr>
<td><a href="coexists_with/">coexists_with</a></td>
<td>holds between two entities that are co-located in the same aggregate object, process, or spatio-temporal region</td>
</tr>
<tr>
<td><a href="coexpressed_with/">coexpressed_with</a></td>
<td>holds between any two genes or gene products, in which both are generally expressed within a single defined experimental context.</td>
</tr>
<tr>
<td><a href="colocalizes_with/">colocalizes_with</a></td>
<td>holds between two entities that are observed to be located in the same place.</td>
</tr>
<tr>
<td><a href="completed_by/">completed_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="composed_primarily_of/">composed_primarily_of</a></td>
<td>x composed_primarily_of_y if:more than half of the mass of x is made from parts of y.</td>
</tr>
<tr>
<td><a href="condition_ameliorated_by/">condition_ameliorated_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="condition_associated_with_gene/">condition_associated_with_gene</a></td>
<td>holds between a gene and a disease or phenotypic feature that may be influenced, contribute to, or be correlated with the gene or its alleles/products</td>
</tr>
<tr>
<td><a href="condition_exacerbated_by/">condition_exacerbated_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="condition_predisposed_by/">condition_predisposed_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="condition_prevented_by/">condition_prevented_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="condition_promoted_by/">condition_promoted_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="consumed_by/">consumed_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="consumes/">consumes</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="contains_process/">contains_process</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="contraindicated_in/">contraindicated_in</a></td>
<td>Holds between a substance, procedure, or activity and a medical condition or circumstance, where an authority has established that the substance, procedure, or activity should not be applied as an intervention in patients with the condition or circumstance because it can result in detrimental outcomes.</td>
</tr>
<tr>
<td><a href="contributes_to/">contributes_to</a></td>
<td>holds between two entities where the occurrence, existence, or activity of one contributes to the occurrence or generation of the other</td>
</tr>
<tr>
<td><a href="contribution_from/">contribution_from</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="contributor/">contributor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="correlated_with/">correlated_with</a></td>
<td>A relationship that holds between two concepts represented by variables for which a statistical correlation is believed to exist, as demonstrated using a correlation analysis method.</td>
</tr>
<tr>
<td><a href="decreased_amount_in/">decreased_amount_in</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="decreased_likelihood_associated_with/">decreased_likelihood_associated_with</a></td>
<td></td>
</tr>
<tr>
<td><a href="decreases_amount_or_activity_of/">decreases_amount_or_activity_of</a></td>
<td>A grouping mixin to help with searching for all the predicates that decrease the amount or activity of the object.</td>
</tr>
<tr>
<td><a href="decreases_sensitivity_to/">decreases_sensitivity_to</a></td>
<td>holds between two chemical entities or genes or gene products where the action or effect of one decreases the susceptibility/sensitivity of a biological entity or system (e.g. an organism, cell, cellular component, macromolecular machine mixin, biological or pathological process) to the other</td>
</tr>
<tr>
<td><a href="derives_from/">derives_from</a></td>
<td>holds between two distinct material entities, the new entity and the old entity, in which the new entity begins to exist when the old entity ceases to exist, and the new entity inherits the significant portion of the matter of the old entity</td>
</tr>
<tr>
<td><a href="derives_into/">derives_into</a></td>
<td>holds between two distinct material entities, the old entity and the new entity, in which the new entity begins to exist when the old entity ceases to exist, and the new entity inherits the significant portion of the matter of the old entity</td>
</tr>
<tr>
<td><a href="develops_from/">develops_from</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="develops_into/">develops_into</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="diagnoses/">diagnoses</a></td>
<td>a relationship that identifies the nature of (an illness or other problem) by examination of the symptoms.</td>
</tr>
<tr>
<td><a href="directly_physically_interacts_with/">directly_physically_interacts_with</a></td>
<td>A causal mechanism mediated by a direct contact between the effector and target entities (this contact may be weak or strong, transient or stable).</td>
</tr>
<tr>
<td><a href="disease_has_basis_in/">disease_has_basis_in</a></td>
<td>A relation that holds between a disease and an entity where the state of the entity has contribution to the disease.</td>
</tr>
<tr>
<td><a href="disease_has_location/">disease_has_location</a></td>
<td>A relationship between a disease and an anatomical entity where the disease has one or more features that are located in that entity.</td>
</tr>
<tr>
<td><a href="disrupted_by/">disrupted_by</a></td>
<td>describes a relationship where the structure, function, or occurrence of one entity is degraded or interfered with by another.</td>
</tr>
<tr>
<td><a href="disrupts/">disrupts</a></td>
<td>describes a relationship where one entity degrades or interferes with the structure, function, or occurrence of another.</td>
</tr>
<tr>
<td><a href="editor/">editor</a></td>
<td>editor of a compiled work such as a book or a periodical (newspaper or an academic journal). Note that in the case of publications which have a containing "published in" node property, the editor association may not be attached directly to the embedded child publication, but only made in between the parent's publication node and the editorial agent of the encompassing publication (e.g. only from the Book referenced by the 'published_in' property of a book chapter Publication node).</td>
</tr>
<tr>
<td><a href="enabled_by/">enabled_by</a></td>
<td>holds between a process and a physical entity, where the physical entity executes the process</td>
</tr>
<tr>
<td><a href="enables/">enables</a></td>
<td>holds between a physical entity and a process, where the physical entity executes the process</td>
</tr>
<tr>
<td><a href="exacerbates_condition/">exacerbates_condition</a></td>
<td>Holds between a substance, procedure, or activity and an existing medical condition (disease or phenotypic</td>
</tr>
<tr>
<td>feature) where the substance, procedure, or activity worsens some or all aspects of the condition.</td>
<td></td>
</tr>
<tr>
<td><a href="exact_match/">exact_match</a></td>
<td>holds between two entities that have strictly equivalent meanings, with a high degree of confidence</td>
</tr>
<tr>
<td><a href="expressed_in/">expressed_in</a></td>
<td>holds between a gene or gene product and an anatomical entity in which it is expressed</td>
</tr>
<tr>
<td><a href="expresses/">expresses</a></td>
<td>holds between an anatomical entity and gene or gene product that is expressed there</td>
</tr>
<tr>
<td><a href="food_component_of/">food_component_of</a></td>
<td>holds between a one or more chemical entities present in food, irrespective of nutritional value (i.e. could also be a contaminant or additive)</td>
</tr>
<tr>
<td><a href="gene_associated_with_condition/">gene_associated_with_condition</a></td>
<td>holds between a gene and a disease or phenotypic feature that the gene or its alleles/products may influence, contribute to, or correlate with</td>
</tr>
<tr>
<td><a href="gene_product_of/">gene_product_of</a></td>
<td>definition x has gene product of y if and only if y is a gene (SO:0000704) that participates in some gene expression process (GO:0010467) where the output of thatf process is either y or something that is ribosomally translated from x</td>
</tr>
<tr>
<td><a href="gene_fusion_with/">gene_fusion_with</a></td>
<td>holds between two independent genes that have fused through translocation, interstitial deletion, or chromosomal inversion to form a new, hybrid gene. Fusion genes are often implicated in various neoplasms and cancers.</td>
</tr>
<tr>
<td><a href="genetic_association/">genetic_association</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="genetic_neighborhood_of/">genetic_neighborhood_of</a></td>
<td>holds between two genes located nearby one another on a chromosome.</td>
</tr>
<tr>
<td><a href="genetically_associated_with/">genetically_associated_with</a></td>
<td></td>
</tr>
<tr>
<td><a href="genetically_interacts_with/">genetically_interacts_with</a></td>
<td>holds between two genes whose phenotypic effects are dependent on each other in some way - such that their combined phenotypic effects are the result of some interaction between the activity of their gene products. Examples include epistasis and synthetic lethality.</td>
</tr>
<tr>
<td><a href="has_active_component/">has_active_component</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_active_ingredient/">has_active_ingredient</a></td>
<td>holds between a drug and a molecular entity in which the latter is a part of the former, and is a biologically active component</td>
</tr>
<tr>
<td><a href="has_adverse_event/">has_adverse_event</a></td>
<td>An untoward medical occurrence in a patient or clinical investigation subject that happens during treatment with a therapeutic agent. Adverse events may be caused by something other than the drug or therapy being given and may include abnormal laboratory finding, symptoms, or diseases temporally associated with the treatment, whether or not considered related to the treatment. Adverse events are unintended effects that occur when a medication is administered correctly.</td>
</tr>
<tr>
<td><a href="has_author/">has_author</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_biomarker/">has_biomarker</a></td>
<td>holds between a disease or phenotypic feature and a measurable chemical entity that is used as an indicator of the presence or state of the disease or feature.</td>
</tr>
<tr>
<td># metabolite</td>
<td></td>
</tr>
<tr>
<td><a href="has_catalyst/">has_catalyst</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_chemical_role/">has_chemical_role</a></td>
<td>A role is particular behaviour which a chemical entity may exhibit.</td>
</tr>
<tr>
<td><a href="has_completed/">has_completed</a></td>
<td>holds between an entity and a process that the entity is capable of and has completed</td>
</tr>
<tr>
<td><a href="has_contraindication/">has_contraindication</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_contributor/">has_contributor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_decreased_amount/">has_decreased_amount</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_editor/">has_editor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_excipient/">has_excipient</a></td>
<td>holds between a drug and a molecular entities in which the latter is a part of the former, and is a biologically inactive component</td>
</tr>
<tr>
<td><a href="has_food_component/">has_food_component</a></td>
<td>holds between food and one or more chemical entities composing it, irrespective of nutritional value (i.e. could also be a contaminant or additive)</td>
</tr>
<tr>
<td><a href="has_frameshift_variant/">has_frameshift_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_gene_product/">has_gene_product</a></td>
<td>holds between a gene and a transcribed and/or translated product generated from it</td>
</tr>
<tr>
<td><a href="has_increased_amount/">has_increased_amount</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_input/">has_input</a></td>
<td>holds between a process and a continuant, where the continuant is an input into the process</td>
</tr>
<tr>
<td><a href="has_manifestation/">has_manifestation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_member/">has_member</a></td>
<td>Defines a mereological relation between a collection and an item.</td>
</tr>
<tr>
<td><a href="has_metabolite/">has_metabolite</a></td>
<td>holds between two molecular entities in which the second one is derived from the first one as a product of metabolism</td>
</tr>
<tr>
<td><a href="has_missense_variant/">has_missense_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_mode_of_inheritance/">has_mode_of_inheritance</a></td>
<td>Relates a disease or phenotypic feature to its observed genetic segregation and assumed associated underlying DNA manifestation (i.e. autosomal, sex or mitochondrial chromosome).</td>
</tr>
<tr>
<td><a href="has_molecular_consequence/">has_molecular_consequence</a></td>
<td>connects a sequence variant to a class describing the molecular consequence. E.g.  SO:0001583</td>
</tr>
<tr>
<td><a href="has_nearby_variant/">has_nearby_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_negative_upstream_actor/">has_negative_upstream_actor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_negative_upstream_or_within_actor/">has_negative_upstream_or_within_actor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_non_coding_variant/">has_non_coding_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_nonsense_variant/">has_nonsense_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_not_completed/">has_not_completed</a></td>
<td>holds between an entity and a process that the entity is capable of, but has not completed</td>
</tr>
<tr>
<td><a href="has_nutrient/">has_nutrient</a></td>
<td>one or more nutrients which are growth factors for a living organism</td>
</tr>
<tr>
<td><a href="has_output/">has_output</a></td>
<td>holds between a process and a continuant, where the continuant is an output of the process</td>
</tr>
<tr>
<td><a href="has_part/">has_part</a></td>
<td>holds between wholes and their parts (material entities or processes)</td>
</tr>
<tr>
<td><a href="has_participant/">has_participant</a></td>
<td>holds between a process and a continuant, where the continuant is somehow involved in the process</td>
</tr>
<tr>
<td><a href="has_phenotype/">has_phenotype</a></td>
<td>holds between a biological entity and a phenotype, where a phenotype is construed broadly as any kind of quality of an organism part, a collection of these qualities, or a change in quality or qualities (e.g. abnormally increased temperature). In SNOMEDCT, disorders with keyword 'characterized by' should translate into this predicate.</td>
</tr>
<tr>
<td><a href="has_plasma_membrane_part/">has_plasma_membrane_part</a></td>
<td>Holds between a cell c and a protein complex or protein p if and only if that cell has as part a plasma_membrane[GO:0005886], and that plasma membrane has p as part.</td>
</tr>
<tr>
<td><a href="has_positive_upstream_actor/">has_positive_upstream_actor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_positive_upstream_or_within_actor/">has_positive_upstream_or_within_actor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_provider/">has_provider</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_publisher/">has_publisher</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_sequence_location/">has_sequence_location</a></td>
<td>holds between two nucleic acid entities when the subject can be localized in sequence coordinates on the object. For example, between an exon and a chromosome/contig.</td>
</tr>
<tr>
<td><a href="has_sequence_variant/">has_sequence_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_side_effect/">has_side_effect</a></td>
<td>An unintended, but predictable, secondary effect shown to be correlated with a therapeutic agent, drug or treatment. Side effects happen at normal, recommended doses or treatments, and are unrelated to the intended purpose of the medication.</td>
</tr>
<tr>
<td><a href="has_splice_site_variant/">has_splice_site_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_substrate/">has_substrate</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_synonymous_variant/">has_synonymous_variant</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_target/">has_target</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_upstream_actor/">has_upstream_actor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_upstream_or_within_actor/">has_upstream_or_within_actor</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="has_variant_part/">has_variant_part</a></td>
<td>holds between a nucleic acid entity and a nucleic acid entity that is a sub-component of it</td>
</tr>
<tr>
<td><a href="homologous_to/">homologous_to</a></td>
<td>holds between two biological entities that have common evolutionary origin</td>
</tr>
<tr>
<td><a href="in_cell_population_with/">in_cell_population_with</a></td>
<td>holds between two genes or gene products that are expressed in the same cell type or population</td>
</tr>
<tr>
<td><a href="in_clinical_trials_for/">in_clinical_trials_for</a></td>
<td>Holds between an intervention and a medical condition, and reports that a clinical trial  is being or has been performed in human patients to test the potential of the intervention to treat the medical condition (e.g. to ameliorate, stabilize, or cure the condition, or to delay, prevent, or reduce the risk of it manifesting in the first place).</td>
</tr>
<tr>
<td><a href="in_complex_with/">in_complex_with</a></td>
<td>holds between two genes or gene products that are part of (or code for products that are part of) in the same macromolecular complex</td>
</tr>
<tr>
<td><a href="in_linkage_disequilibrium_with/">in_linkage_disequilibrium_with</a></td>
<td>holds between two sequence variants, the presence of which are correlated in a population</td>
</tr>
<tr>
<td><a href="in_pathway_with/">in_pathway_with</a></td>
<td>holds between two genes or gene products that are part of in the same biological pathway</td>
</tr>
<tr>
<td><a href="in_preclinical_trials_for/">in_preclinical_trials_for</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that a pre-clinical study has been performed specifically to test the potential of the  substance, procedure, or activity to treat the medical condition  (i.e. to ameliorate, stabilize, or cure the condition, or to delay, prevent, or reduce the risk of it manifesting in the first place).</td>
</tr>
<tr>
<td><a href="in_taxon/">in_taxon</a></td>
<td>connects an entity to its taxonomic classification. Only certain kinds of entities can be taxonomically classified; see 'thing with taxon'</td>
</tr>
<tr>
<td><a href="increased_amount_of/">increased_amount_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="increased_likelihood_associated_with/">increased_likelihood_associated_with</a></td>
<td></td>
</tr>
<tr>
<td><a href="increases_amount_or_activity_of/">increases_amount_or_activity_of</a></td>
<td>A grouping mixin to help with searching for all the predicates that increase the amount or activity of the object.</td>
</tr>
<tr>
<td><a href="increases_sensitivity_to/">increases_sensitivity_to</a></td>
<td>holds between two chemical entities or genes or gene products where the action or effect of one increases the susceptibility/sensitivity of a biological entity or system (e.g. an organism, cell, cellular component, macromolecular machine mixin, biological or pathological process) to the other</td>
</tr>
<tr>
<td><a href="indirectly_physically_interacts_with/">indirectly_physically_interacts_with</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="interacts_with/">interacts_with</a></td>
<td>holds between any two entities that directly or indirectly interact with each other</td>
</tr>
<tr>
<td><a href="is_active_ingredient_of/">is_active_ingredient_of</a></td>
<td>holds between a molecular entity and a drug, in which the former is a part of the latter, and is a biologically active component</td>
</tr>
<tr>
<td><a href="is_diagnosed_by/">is_diagnosed_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="is_excipient_of/">is_excipient_of</a></td>
<td>holds between a molecular entity and a drug in which the former is a part of the latter, and is a biologically inactive component</td>
</tr>
<tr>
<td><a href="is_frameshift_variant_of/">is_frameshift_variant_of</a></td>
<td>holds between a sequence variant and a gene, such the sequence variant causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three.</td>
</tr>
<tr>
<td><a href="is_input_of/">is_input_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="is_metabolite_of/">is_metabolite_of</a></td>
<td>holds between two molecular entities in which the first one is derived from the second one as a product of metabolism</td>
</tr>
<tr>
<td><a href="is_missense_variant_of/">is_missense_variant_of</a></td>
<td>holds between a gene  and a sequence variant, such the sequence variant results in a different amino acid sequence but where the length is preserved.</td>
</tr>
<tr>
<td><a href="is_molecular_consequence_of/">is_molecular_consequence_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="is_nearby_variant_of/">is_nearby_variant_of</a></td>
<td>holds between a sequence variant and a gene sequence that the variant is genomically close to.</td>
</tr>
<tr>
<td><a href="is_non_coding_variant_of/">is_non_coding_variant_of</a></td>
<td>holds between a sequence variant and a gene, where the variant does not affect the coding sequence</td>
</tr>
<tr>
<td><a href="is_nonsense_variant_of/">is_nonsense_variant_of</a></td>
<td>holds between a sequence variant and a gene, such the sequence variant results in a premature stop codon</td>
</tr>
<tr>
<td><a href="is_output_of/">is_output_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="is_sequence_variant_of/">is_sequence_variant_of</a></td>
<td>holds between a sequence variant and a nucleic acid entity</td>
</tr>
<tr>
<td><a href="is_side_effect_of/">is_side_effect_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="is_splice_site_variant_of/">is_splice_site_variant_of</a></td>
<td>holds between a sequence variant and a gene, such the sequence variant is in the canonical splice site of one of the gene's exons.</td>
</tr>
<tr>
<td><a href="is_substrate_of/">is_substrate_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="is_synonymous_variant_of/">is_synonymous_variant_of</a></td>
<td>holds between a sequence variant and a gene, such the sequence variant is in the coding sequence of the gene, but results in the same amino acid sequence</td>
</tr>
<tr>
<td><a href="lacks_part/">lacks_part</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="likelihood_affected_by/">likelihood_affected_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="likelihood_associated_with/">likelihood_associated_with</a></td>
<td></td>
</tr>
<tr>
<td><a href="located_in/">located_in</a></td>
<td>holds between a material entity and a material entity or site within which it is located (but of which it is not considered a part)</td>
</tr>
<tr>
<td><a href="location_of/">location_of</a></td>
<td>holds between material entity or site and a material entity that is located within it (but not considered a part of it)</td>
</tr>
<tr>
<td><a href="location_of_disease/">location_of_disease</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="manifestation_of/">manifestation_of</a></td>
<td>that part of a phenomenon which is directly observable or visibly expressed, or which gives evidence to the underlying process; used in SemMedDB for linking things like dysfunctions and processes to some disease or syndrome</td>
</tr>
<tr>
<td><a href="member_of/">member_of</a></td>
<td>Defines a mereological relation between a item and a collection.</td>
</tr>
<tr>
<td><a href="mentioned_by/">mentioned_by</a></td>
<td>refers to is a relation between one named thing and the information content entity that it makes reference to.</td>
</tr>
<tr>
<td><a href="mentions/">mentions</a></td>
<td>refers to is a relation between one information content entity and the named thing that it makes reference to.</td>
</tr>
<tr>
<td><a href="missing_from/">missing_from</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="mode_of_inheritance_of/">mode_of_inheritance_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="model_of/">model_of</a></td>
<td>holds between a thing and some other thing it approximates for purposes of scientific study, in virtue of its exhibiting similar features of the studied entity.</td>
</tr>
<tr>
<td><a href="models/">models</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="models_demonstrating_benefits_for/">models_demonstrating_benefits_for</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="narrow_match/">narrow_match</a></td>
<td>a list of terms from different schemas or terminology systems that have a narrower, more specific meaning. Narrower terms are typically shown as children in a hierarchy or tree.</td>
</tr>
<tr>
<td><a href="negatively_correlated_with/">negatively_correlated_with</a></td>
<td>A relationship that holds between two concepts represented by variables for which a statistical correlation is demonstrated, wherein variable values move in opposite directions (i.e. increased in one or presence of one correlates with a decrease or absence of the other).</td>
</tr>
<tr>
<td><a href="not_completed_by/">not_completed_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="nutrient_of/">nutrient_of</a></td>
<td>holds between a one or more chemical entities present in food, irrespective of nutritional value (i.e. could also be a contaminant or additive)</td>
</tr>
<tr>
<td><a href="occurs_in/">occurs_in</a></td>
<td>holds between a process and a material entity or site within which the process occurs</td>
</tr>
<tr>
<td><a href="occurs_in_disease/">occurs_in_disease</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="occurs_together_in_literature_with/">occurs_together_in_literature_with</a></td>
<td>holds between two entities where their co-occurrence is correlated by counts of publications in which both occur, using some threshold of occurrence as defined by the edge provider.</td>
</tr>
<tr>
<td><a href="opposite_of/">opposite_of</a></td>
<td>x is the opposite of y if there exists some distance metric M, and there exists no z such as M(x,z) &lt;= M(x,y) or M(y,z) &lt;= M(y,x). (This description is from RO. Needs to be rephrased).</td>
</tr>
<tr>
<td><a href="orthologous_to/">orthologous_to</a></td>
<td>a homology relationship between entities (typically genes) that diverged after a speciation event.</td>
</tr>
<tr>
<td><a href="overlaps/">overlaps</a></td>
<td>holds between entities that overlap in their extents (materials or processes)</td>
</tr>
<tr>
<td><a href="paralogous_to/">paralogous_to</a></td>
<td>a homology relationship that holds between entities (typically genes) that diverged after a duplication event.</td>
</tr>
<tr>
<td><a href="part_of/">part_of</a></td>
<td>holds between parts and wholes (material entities or processes)</td>
</tr>
<tr>
<td><a href="participates_in/">participates_in</a></td>
<td>holds between a continuant and a process, where the continuant is somehow involved in the process</td>
</tr>
<tr>
<td><a href="phenotype_of/">phenotype_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="physically_interacts_with/">physically_interacts_with</a></td>
<td>holds between two entities that make physical contact as part of some interaction. does not imply a causal relationship.</td>
</tr>
<tr>
<td><a href="plasma_membrane_part_of/">plasma_membrane_part_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="positively_correlated_with/">positively_correlated_with</a></td>
<td>A relationship that holds between two concepts represented by variables for which a statistical correlation is demonstrated, wherein variable values move together in the same direction (i.e. increased in one or presence of one correlates with an increase or presence of the other).</td>
</tr>
<tr>
<td><a href="preceded_by/">preceded_by</a></td>
<td>holds between two processes, where the other is completed before the one begins</td>
</tr>
<tr>
<td><a href="precedes/">precedes</a></td>
<td>holds between two processes, where one completes before the other begins</td>
</tr>
<tr>
<td><a href="predisposes_to_condition/">predisposes_to_condition</a></td>
<td>Holds between two entities where the presence or application of one increases the chance that the other will come to be.</td>
</tr>
<tr>
<td><a href="preventative_for_condition/">preventative_for_condition</a></td>
<td>Holds between a substance, procedure, or activity and a medical condition (disease or phenotypic feature), and states that the  substance, procedure, or activity is able to prevent it manifesting in the first place.</td>
</tr>
<tr>
<td><a href="primarily_composed_of/">primarily_composed_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="produced_by/">produced_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="produces/">produces</a></td>
<td>holds between a material entity and a product that is generated through the intentional actions or functioning of the material entity</td>
</tr>
<tr>
<td><a href="promotes_condition/">promotes_condition</a></td>
<td>Holds between a substance, procedure, or activity and a medical condition (disease or phenotypic feature), and states that the  substance, procedure, or activity is able to promote it manifesting in the first place.</td>
</tr>
<tr>
<td><a href="provider/">provider</a></td>
<td>person, group, organization or project that provides a piece of information (e.g. a knowledge association).</td>
</tr>
<tr>
<td><a href="publisher/">publisher</a></td>
<td>organization or person responsible for publishing books, periodicals, podcasts, games or software. Note that in the case of publications which have a containing "published in" node property, the publisher association may not be attached directly to the embedded child publication, but only made in between the parent's publication node and the publisher agent of the encompassing publication (e.g. only from the Journal referenced by the 'published_in' property of an journal article Publication node).</td>
</tr>
<tr>
<td><a href="regulated_by/">regulated_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="regulates/">regulates</a></td>
<td>A more specific form of affects, that implies the effect results from a biologically evolved control mechanism. Gene-affects-gene relationships will (almost) always involve regulation.  Exogenous/environmental chemical-affects-gene relationships are not cases of regulation in this definition. Instead these would be captured using the 'affects' predicate, or possibly one of the 'interacts with' predicates depending on the nature of the interaction.</td>
</tr>
<tr>
<td><a href="related_condition/">related_condition</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="related_to/">related_to</a></td>
<td>A relationship that is asserted between two named things</td>
</tr>
<tr>
<td><a href="related_to_at_concept_level/">related_to_at_concept_level</a></td>
<td>Represents a relationship held between terminology components that describe the conceptual model of a domain.</td>
</tr>
<tr>
<td><a href="related_to_at_instance_level/">related_to_at_instance_level</a></td>
<td>Represents a relationship held between two instances of a data classes.  Much like an assertion component, in an ABox, these represent facts associated with the conceptual model.</td>
</tr>
<tr>
<td><a href="resistance_associated_with/">resistance_associated_with</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="response_associated_with/">response_associated_with</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="same_as/">same_as</a></td>
<td>holds between two entities that are considered equivalent to each other</td>
</tr>
<tr>
<td><a href="sensitivity_affected_by/">sensitivity_affected_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="sensitivity_associated_with/">sensitivity_associated_with</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="sensitivity_decreased_by/">sensitivity_decreased_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="sensitivity_increased_by/">sensitivity_increased_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="sequence_location_of/">sequence_location_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="similar_to/">similar_to</a></td>
<td>holds between an entity and some other entity with similar features.</td>
</tr>
<tr>
<td><a href="studied_to_treat/">studied_to_treat</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that one or more scientific study has been performed to specifically test the potential of the  substance, procedure, or activity to treat the medical condition  (i.e. to ameliorate, stabilize, or cure the condition, or to delay, prevent, or reduce the risk of it manifesting in the first place).</td>
</tr>
<tr>
<td><a href="subclass_of/">subclass_of</a></td>
<td>holds between two classes where the domain class is a specialization of the range class</td>
</tr>
<tr>
<td><a href="subject_of_treatment_application_or_study_for_treatment_by/">subject_of_treatment_application_or_study_for_treatment_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="superclass_of/">superclass_of</a></td>
<td>holds between two classes where the domain class is a super class of the range class</td>
</tr>
<tr>
<td><a href="target_for/">target_for</a></td>
<td>A gene is a target of a disease when its products are druggable and when a drug interaction with the gene product could have a therapeutic effect</td>
</tr>
<tr>
<td><a href="taxon_of/">taxon_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="temporally_related_to/">temporally_related_to</a></td>
<td>holds between two entities with a temporal relationship</td>
</tr>
<tr>
<td><a href="tested_by_clinical_trials_of/">tested_by_clinical_trials_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="tested_by_preclinical_trials_of/">tested_by_preclinical_trials_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="transcribed_from/">transcribed_from</a></td>
<td>x is transcribed from y if and only if x is synthesized from template y</td>
</tr>
<tr>
<td><a href="transcribed_to/">transcribed_to</a></td>
<td>inverse of transcribed from</td>
</tr>
<tr>
<td><a href="translates_to/">translates_to</a></td>
<td>x (amino acid chain/polypeptide) is the ribosomal translation of y (transcript) if and only if a ribosome reads y (transcript) through a series of triplet codon-amino acid adaptor activities (GO:0030533) and produces x (amino acid chain/polypeptide)</td>
</tr>
<tr>
<td><a href="translation_of/">translation_of</a></td>
<td>inverse of translates to</td>
</tr>
<tr>
<td><a href="treated_by/">treated_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="treated_in_studies_by/">treated_in_studies_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="treatment_applications_from/">treatment_applications_from</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="treats/">treats</a></td>
<td>Holds between an intervention (substance, procedure, or activity) and a medical condition (disease or phenotypic feature), and states that the intervention is, in some population(s), able to ameliorate, stabilize, or cure the condition or delay, prevent, or reduce the risk of it manifesting in the first place. ‘Treats’ edges should be asserted (knowledge_level: assertion) only in cases where there is strong supporting evidence - i.e. in some population(s) the intervention is approved for the condition, passed phase 3 or in phase 4 trials for the condition, or is an otherwise established treatment in the medical community (e.g. a widely-accepted or formally recommended off-label use). In the absence of such evidence, weaker predicates should be used in asserted edges (e.g. ‘in clinical trials for’ or ‘beneficial in models of’). ‘Treats’ edges based on weaker or indirect forms of evidence can however be created as predictions (knowledge_level: prediction) and should point to the more foundational asserted edges that support them.</td>
</tr>
<tr>
<td><a href="treats_or_applied_or_studied_to_treat/">treats_or_applied_or_studied_to_treat</a></td>
<td>Holds between an substance, procedure, or activity and a medical condition (disease or phenotypic feature), and states that the substance, procedure, or activity is able to treat the condition, has been observed to be taken/prescribed in practice with the intent of treating the condition, or has been interrogated in a scientific study that hypothesized an ability to treat the condition (in humans or other biological systems/organisms).</td>
</tr>
<tr>
<td><a href="variant_part_of/">variant_part_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="was_tested_for_effect_of/">was_tested_for_effect_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="was_tested_for_effect_on/">was_tested_for_effect_on</a></td>
<td>Reports that the subject was interrogated in an experiment to determine how it may affect the object. A relationship between some perturbing agent (usually a chemical compound) and some target entity, where the affect of the perturbing agent on the target entity was interrogated in a particular assay. The target might be a particular protein, tissue, phenotype, whole organism, cell line, or other type of biological entity.</td>
</tr>
<tr>
<td><a href="xenologous_to/">xenologous_to</a></td>
<td>a homology relationship characterized by an interspecies (horizontal) transfer since the common ancestor.</td>
</tr>
</tbody>
</table>
<h2 id="qualifiers-visualization">Qualifiers (<a href="qualifiers.html">Visualization</a>)</h2>
<table>
<thead>
<tr>
<th>Property</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="anatomical_context_qualifier/">anatomical_context_qualifier</a></td>
<td>A statement qualifier representing an anatomical location where an relationship expressed in an association took place (can be a tissue, cell type, or sub-cellular location).</td>
</tr>
<tr>
<td><a href="aspect_qualifier/">aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.</td>
</tr>
<tr>
<td><a href="causal_mechanism_qualifier/">causal_mechanism_qualifier</a></td>
<td>A statement qualifier representing a type of molecular control mechanism through which an effect of a chemical on a gene or gene product is mediated (e.g. 'agonism', 'inhibition', 'allosteric modulation', 'channel blocker')</td>
</tr>
<tr>
<td><a href="context_qualifier/">context_qualifier</a></td>
<td>Restricts the setting/context/location where the core concept (or qualified core concept) resides or occurs.</td>
</tr>
<tr>
<td><a href="derivative_qualifier/">derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.</td>
</tr>
<tr>
<td><a href="direction_qualifier/">direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree.</td>
</tr>
<tr>
<td><a href="disease_context_qualifier/">disease_context_qualifier</a></td>
<td>A context qualifier representing a disease or condition in which a relationship expressed in an association took place.</td>
</tr>
<tr>
<td><a href="form_or_variant_qualifier/">form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.</td>
</tr>
<tr>
<td><a href="frequency_qualifier/">frequency_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state how frequent the phenotype is observed in the subject</td>
</tr>
<tr>
<td><a href="object_aspect_qualifier/">object_aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.  This qualifier specifies a change in the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="object_context_qualifier/">object_context_qualifier</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="object_derivative_qualifier/">object_derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.  This qualifier is for the object of an association (or statement).</td>
</tr>
<tr>
<td><a href="object_direction_qualifier/">object_direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree. This qualifier qualifies the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="object_form_or_variant_qualifier/">object_form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.  This qualifier specifies a change in the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="object_part_qualifier/">object_part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).  This qualifier is for the object of an association (or statement).</td>
</tr>
<tr>
<td><a href="object_specialization_qualifier/">object_specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="onset_qualifier/">onset_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state when the phenotype appears is in the subject.</td>
</tr>
<tr>
<td><a href="part_qualifier/">part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).</td>
</tr>
<tr>
<td><a href="population_context_qualifier/">population_context_qualifier</a></td>
<td>a biological population (general, study, cohort, etc.) with a specific set of characteristics to constrain an association.</td>
</tr>
<tr>
<td><a href="qualified_predicate/">qualified_predicate</a></td>
<td>Predicate to be used in an association when subject and object qualifiers are present and the full reading of the statement requires a qualification to the predicate in use in order to refine or increase the specificity of the full statement reading.  This qualifier holds a relationship to be used instead of that expressed by the primary predicate, in a ‘full statement’ reading of the association, where qualifier-based semantics are included.  This is necessary only in cases where the primary predicate does not work in a full statement reading.</td>
</tr>
<tr>
<td><a href="qualifier/">qualifier</a></td>
<td>grouping slot for all qualifiers on an edge.  useful for testing compliance with association classes</td>
</tr>
<tr>
<td><a href="response_context_qualifier/">response_context_qualifier</a></td>
<td>a biological response (general, study, cohort, etc.) with a specific set of characteristics to constrain an association.</td>
</tr>
<tr>
<td><a href="response_target_context_qualifier/">response_target_context_qualifier</a></td>
<td>a biological response target (a patient, a cohort, a model system, a cell line, a sample of biological material, etc.)</td>
</tr>
<tr>
<td><a href="severity_qualifier/">severity_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state how severe the phenotype is in the subject</td>
</tr>
<tr>
<td><a href="sex_qualifier/">sex_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state whether the association is specific to a particular sex.</td>
</tr>
<tr>
<td><a href="specialization_qualifier/">specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="species_context_qualifier/">species_context_qualifier</a></td>
<td>A statement qualifier representing a taxonomic category of species in which a relationship expressed in an association took place.</td>
</tr>
<tr>
<td><a href="stage_qualifier/">stage_qualifier</a></td>
<td>stage during which gene or protein expression of takes place.</td>
</tr>
<tr>
<td><a href="statement_qualifier/">statement_qualifier</a></td>
<td>A property that qualifies the entirety of the statement made in an association.  It applies to both a fully qualified subject and a fully qualified object as well as the predicate and qualified predicate in an association.</td>
</tr>
<tr>
<td><a href="subject_aspect_qualifier/">subject_aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.  This qualifier specifies a change in the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="subject_context_qualifier/">subject_context_qualifier</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="subject_derivative_qualifier/">subject_derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.  This qualifier is for the subject of an association (or statement).</td>
</tr>
<tr>
<td><a href="subject_direction_qualifier/">subject_direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree. This qualifier qualifies the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="subject_form_or_variant_qualifier/">subject_form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.  This qualifier specifies a change in the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="subject_part_qualifier/">subject_part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).  This qualifier is for the subject of an association (or statement).</td>
</tr>
<tr>
<td><a href="subject_specialization_qualifier/">subject_specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="temporal_context_qualifier/">temporal_context_qualifier</a></td>
<td>a constraint of time placed upon the truth value of an association. for time intervales, use temporal interval qualifier.</td>
</tr>
<tr>
<td><a href="temporal_interval_qualifier/">temporal_interval_qualifier</a></td>
<td>a constraint of a time interval placed upon the truth value of an association.</td>
</tr>
</tbody>
</table>
<h2 id="associations">Associations</h2>
<table>
<thead>
<tr>
<th>Association</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AnatomicalEntityToAnatomicalEntityAssociation/">AnatomicalEntityToAnatomicalEntityAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="AnatomicalEntityToAnatomicalEntityOntogenicAssociation/">AnatomicalEntityToAnatomicalEntityOntogenicAssociation</a></td>
<td>A relationship between two anatomical entities where the relationship is ontogenic, i.e. the two entities are related by development. A number of different relationship types can be used to specify the precise nature of the relationship.</td>
</tr>
<tr>
<td><a href="AnatomicalEntityToAnatomicalEntityPartOfAssociation/">AnatomicalEntityToAnatomicalEntityPartOfAssociation</a></td>
<td>A relationship between two anatomical entities where the relationship is mereological, i.e the two entities are related by parthood. This includes relationships between cellular components and cells, between cells and tissues, tissues and whole organisms</td>
</tr>
<tr>
<td><a href="Association/">Association</a></td>
<td>A typed association between two entities, supported by evidence</td>
</tr>
<tr>
<td><a href="BehaviorToBehavioralFeatureAssociation/">BehaviorToBehavioralFeatureAssociation</a></td>
<td>An association between an mixture behavior and a behavioral feature manifested by the individual exhibited or has exhibited the behavior.</td>
</tr>
<tr>
<td><a href="BiologicalProcessOrActivityToAnatomicalEntityAssociation/">BiologicalProcessOrActivityToAnatomicalEntityAssociation</a></td>
<td>An association between a biological process or activity and an anatomical entity.</td>
</tr>
<tr>
<td><a href="BiologicalProcessOrActivityToBiologicalProcessOrActivityAssociation/">BiologicalProcessOrActivityToBiologicalProcessOrActivityAssociation</a></td>
<td>Classification relationship between biological processes or activities (e.g. coupling of two molecular activities;  assignment of molecular activity to a pathway; implicating a pathway in a biological process; etc.)</td>
</tr>
<tr>
<td><a href="BiologicalProcessOrActivityToGeneOrGeneProductOrGeneFamilyAssociation/">BiologicalProcessOrActivityToGeneOrGeneProductOrGeneFamilyAssociation</a></td>
<td>Relationship between a biological processor activity (e.g. molecular activity, biological process or pathway) to gene or gene product or gene family.</td>
</tr>
<tr>
<td><a href="CaseToDiseaseAssociation/">CaseToDiseaseAssociation</a></td>
<td>An association between a Case (patient) and a Disease</td>
</tr>
<tr>
<td><a href="CaseToGeneAssociation/">CaseToGeneAssociation</a></td>
<td>Association between a Case and a Gene (e.g., indicating a gene of interest for the case)</td>
</tr>
<tr>
<td><a href="CaseToPhenotypicFeatureAssociation/">CaseToPhenotypicFeatureAssociation</a></td>
<td>An association between a case (e.g. individual patient) and a phenotypic feature in which the individual has or has had the phenotype.</td>
</tr>
<tr>
<td><a href="CaseToVariantAssociation/">CaseToVariantAssociation</a></td>
<td>Association between a Case and a Genetic Variant</td>
</tr>
<tr>
<td><a href="CausalGeneToDiseaseAssociation/">CausalGeneToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="CellLineAsAModelOfDiseaseAssociation/">CellLineAsAModelOfDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="CellLineToDiseaseOrPhenotypicFeatureAssociation/">CellLineToDiseaseOrPhenotypicFeatureAssociation</a></td>
<td>An relationship between a cell line and a disease or a phenotype, where the cell line is derived from an individual with that disease or phenotype.</td>
</tr>
<tr>
<td><a href="ChemicalAffectsBiologicalEntityAssociation/">ChemicalAffectsBiologicalEntityAssociation</a></td>
<td>Describes an effect that a chemical has on a biological entity (e.g. an impact of on its abundance, activity,localization, processing, expression, etc.)</td>
</tr>
<tr>
<td><a href="ChemicalAffectsGeneAssociation/">ChemicalAffectsGeneAssociation</a></td>
<td>Describes an effect that a chemical has on a gene or gene product (e.g. an impact of on its abundance, activity,localization, processing, expression, etc.)</td>
</tr>
<tr>
<td><a href="ChemicalEntityAssessesNamedThingAssociation/">ChemicalEntityAssessesNamedThingAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ChemicalEntityOrGeneOrGeneProductRegulatesGeneAssociation/">ChemicalEntityOrGeneOrGeneProductRegulatesGeneAssociation</a></td>
<td>A regulatory relationship between two genes</td>
</tr>
<tr>
<td><a href="ChemicalGeneInteractionAssociation/">ChemicalGeneInteractionAssociation</a></td>
<td>describes a physical interaction between a chemical entity and a gene or gene product. Any biological or chemical effect resulting from such an interaction are out of scope, and covered by the ChemicalAffectsGeneAssociation type (e.g. impact of a chemical on the abundance, activity, structure, etc, of either participant in the interaction)</td>
</tr>
<tr>
<td><a href="ChemicalOrDrugOrTreatmentSideEffectDiseaseOrPhenotypicFeatureAssociation/">ChemicalOrDrugOrTreatmentSideEffectDiseaseOrPhenotypicFeatureAssociation</a></td>
<td>This association defines a relationship between a chemical or treatment (or procedure) and a disease or phenotypic feature where the disesae or phenotypic feature is a secondary, typically (but not always) undesirable effect.</td>
</tr>
<tr>
<td><a href="ChemicalOrDrugOrTreatmentToDiseaseOrPhenotypicFeatureAssociation/">ChemicalOrDrugOrTreatmentToDiseaseOrPhenotypicFeatureAssociation</a></td>
<td>This association defines a relationship between a chemical or treatment (or procedure) and a disease or phenotypic feature where the disease or phenotypic feature is a secondary undesirable effect.</td>
</tr>
<tr>
<td><a href="ChemicalToChemicalAssociation/">ChemicalToChemicalAssociation</a></td>
<td>A relationship between two chemical entities. This can encompass actual interactions as well as temporal causal edges, e.g. one chemical converted to another.</td>
</tr>
<tr>
<td><a href="ChemicalToChemicalDerivationAssociation/">ChemicalToChemicalDerivationAssociation</a></td>
<td>A causal relationship between two chemical entities, where the subject represents the upstream entity and the object represents the downstream. For any such association there is an implicit reaction: IF R has-input C1 AND R has-output C2 AND R enabled-by P AND R type Reaction THEN C1 derives-into C2 catalyst qualifier P</td>
</tr>
<tr>
<td><a href="ChemicalToDiseaseOrPhenotypicFeatureAssociation/">ChemicalToDiseaseOrPhenotypicFeatureAssociation</a></td>
<td>An interaction between a chemical entity and a phenotype or disease, where the presence of the chemical gives rise to or exacerbates the phenotype.</td>
</tr>
<tr>
<td><a href="ChemicalToPathwayAssociation/">ChemicalToPathwayAssociation</a></td>
<td>An interaction between a chemical entity and a biological process or pathway.</td>
</tr>
<tr>
<td><a href="ContributorAssociation/">ContributorAssociation</a></td>
<td>Any association between an entity (such as a publication) and various agents that contribute to its realisation</td>
</tr>
<tr>
<td><a href="CorrelatedGeneToDiseaseAssociation/">CorrelatedGeneToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="DiseaseAssociatedWithResponseToChemicalEntityAssociation/">DiseaseAssociatedWithResponseToChemicalEntityAssociation</a></td>
<td>A statistical association between a disease and a chemical entity where the chemical entity has a therapeutic or adverse effect on the disease progression, symptoms or outcomes in a patient, cell line, or any model system.</td>
</tr>
<tr>
<td><a href="DiseaseOrPhenotypicFeatureToGeneticInheritanceAssociation/">DiseaseOrPhenotypicFeatureToGeneticInheritanceAssociation</a></td>
<td>An association between either a disease or a phenotypic feature and its mode of (genetic) inheritance.</td>
</tr>
<tr>
<td><a href="DiseaseOrPhenotypicFeatureToLocationAssociation/">DiseaseOrPhenotypicFeatureToLocationAssociation</a></td>
<td>An association between either a disease or a phenotypic feature and an anatomical entity, where the disease/feature manifests in that site.</td>
</tr>
<tr>
<td><a href="DiseaseToExposureEventAssociation/">DiseaseToExposureEventAssociation</a></td>
<td>An association between an exposure event and a disease.</td>
</tr>
<tr>
<td><a href="DiseaseToPhenotypicFeatureAssociation/">DiseaseToPhenotypicFeatureAssociation</a></td>
<td>An association between a disease and a phenotypic feature in which the phenotypic feature is associated with the disease in some way.</td>
</tr>
<tr>
<td><a href="DrugToGeneAssociation/">DrugToGeneAssociation</a></td>
<td>An interaction between a drug and a gene or gene product.</td>
</tr>
<tr>
<td><a href="DruggableGeneToDiseaseAssociation/">DruggableGeneToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="EntityToDiseaseAssociation/">EntityToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="EntityToPhenotypicFeatureAssociation/">EntityToPhenotypicFeatureAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ExonToTranscriptRelationship/">ExonToTranscriptRelationship</a></td>
<td>A transcript is formed from multiple exons</td>
</tr>
<tr>
<td><a href="ExposureEventToOutcomeAssociation/">ExposureEventToOutcomeAssociation</a></td>
<td>An association between an exposure event and an outcome.</td>
</tr>
<tr>
<td><a href="ExposureEventToPhenotypicFeatureAssociation/">ExposureEventToPhenotypicFeatureAssociation</a></td>
<td>Any association between an environment and a phenotypic feature, where being in the environment influences the phenotype.</td>
</tr>
<tr>
<td><a href="FunctionalAssociation/">FunctionalAssociation</a></td>
<td>An association between a macromolecular machine mixin (gene, gene product or complex of gene products) and either a molecular activity, a biological process or a cellular location in which a function is executed.</td>
</tr>
<tr>
<td><a href="GeneAffectsChemicalAssociation/">GeneAffectsChemicalAssociation</a></td>
<td>Describes an effect that a gene or gene product has on a chemical entity (e.g. an impact of on its abundance, activity, localization, processing, transport, etc.)</td>
</tr>
<tr>
<td><a href="GeneAsAModelOfDiseaseAssociation/">GeneAsAModelOfDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GeneFamilyToGeneOrGeneProductOrGeneFamilyAssociation/">GeneFamilyToGeneOrGeneProductOrGeneFamilyAssociation</a></td>
<td>Relationship between a gene family and a contained gene or gene product or gene family.</td>
</tr>
<tr>
<td><a href="GeneHasVariantThatContributesToDiseaseAssociation/">GeneHasVariantThatContributesToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GeneOrGeneProductOrGeneFamilyToAnatomicalEntityAssociation/">GeneOrGeneProductOrGeneFamilyToAnatomicalEntityAssociation</a></td>
<td>An association between a gene or gene product or gene family and an anatomical entity.</td>
</tr>
<tr>
<td><a href="GeneOrGeneProductOrGeneFamilyToBiologicalProcessOrActivityAssociation/">GeneOrGeneProductOrGeneFamilyToBiologicalProcessOrActivityAssociation</a></td>
<td>Relationship between a gene or gene product or gene family to a specified biological process or activity (e.g. molecular activity, biological process or pathway).</td>
</tr>
<tr>
<td><a href="GeneRegulatesGeneAssociation/">GeneRegulatesGeneAssociation</a></td>
<td>Describes a regulatory relationship between two genes or gene products.</td>
</tr>
<tr>
<td><a href="GeneToDiseaseAssociation/">GeneToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GeneToDiseaseOrPhenotypicFeatureAssociation/">GeneToDiseaseOrPhenotypicFeatureAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GeneToExpressionSiteAssociation/">GeneToExpressionSiteAssociation</a></td>
<td>An association between a gene and a gene expression site, possibly qualified by stage/timing info.</td>
</tr>
<tr>
<td><a href="GeneToGeneAssociation/">GeneToGeneAssociation</a></td>
<td>abstract parent class for different kinds of gene-gene or gene product to gene product relationships. Includes homology and interaction.</td>
</tr>
<tr>
<td><a href="GeneToGeneCoexpressionAssociation/">GeneToGeneCoexpressionAssociation</a></td>
<td>Indicates that two genes are co-expressed, generally under the same conditions.</td>
</tr>
<tr>
<td><a href="GeneToGeneFamilyAssociation/">GeneToGeneFamilyAssociation</a></td>
<td>Set membership of a gene in a family of genes related by common evolutionary ancestry usually inferred by sequence comparisons. The genes in a given family generally share common sequence motifs which generally map onto shared gene product structure-function relationships.</td>
</tr>
<tr>
<td><a href="GeneToGeneHomologyAssociation/">GeneToGeneHomologyAssociation</a></td>
<td>A homology association between two genes. May be orthology (in which case the species of subject and object should differ) or paralogy (in which case the species may be the same)</td>
</tr>
<tr>
<td><a href="GeneToGeneProductRelationship/">GeneToGeneProductRelationship</a></td>
<td>A gene is transcribed and potentially translated to a gene product</td>
</tr>
<tr>
<td><a href="GeneToGoTermAssociation/">GeneToGoTermAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GeneToPathwayAssociation/">GeneToPathwayAssociation</a></td>
<td>An interaction between a gene or gene product and a biological process or pathway.</td>
</tr>
<tr>
<td><a href="GeneToPhenotypicFeatureAssociation/">GeneToPhenotypicFeatureAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GenomicSequenceLocalization/">GenomicSequenceLocalization</a></td>
<td>A relationship between a sequence feature and a nucleic acid entity it is localized to. The reference entity may be a chromosome, chromosome region or information entity such as a contig.</td>
</tr>
<tr>
<td><a href="GenotypeAsAModelOfDiseaseAssociation/">GenotypeAsAModelOfDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GenotypeToDiseaseAssociation/">GenotypeToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GenotypeToGeneAssociation/">GenotypeToGeneAssociation</a></td>
<td>Any association between a genotype and a gene. The genotype have have multiple variants in that gene or a single one. There is no assumption of cardinality</td>
</tr>
<tr>
<td><a href="GenotypeToGenotypePartAssociation/">GenotypeToGenotypePartAssociation</a></td>
<td>Any association between one genotype and a genotypic entity that is a sub-component of it</td>
</tr>
<tr>
<td><a href="GenotypeToPhenotypicFeatureAssociation/">GenotypeToPhenotypicFeatureAssociation</a></td>
<td>Any association between one genotype and a phenotypic feature, where having the genotype confers the phenotype, either in isolation or through environment</td>
</tr>
<tr>
<td><a href="GenotypeToVariantAssociation/">GenotypeToVariantAssociation</a></td>
<td>Any association between a genotype and a sequence variant.</td>
</tr>
<tr>
<td><a href="InformationContentEntityToNamedThingAssociation/">InformationContentEntityToNamedThingAssociation</a></td>
<td>association between a named thing and a information content entity where the specific context of the relationship between that named thing and the publication is unknown. For example, model organisms databases often capture the knowledge that a gene is found in a journal article, but not specifically the context in which that gene was documented in the article. In these cases, this association with the accompanying predicate 'mentions' could be used. Conversely, for more specific associations (like 'gene to disease association', the publication should be captured as an edge property).</td>
</tr>
<tr>
<td><a href="MacromolecularMachineToBiologicalProcessAssociation/">MacromolecularMachineToBiologicalProcessAssociation</a></td>
<td>A functional association between a macromolecular machine (gene, gene product or complex) and a biological process or pathway (as represented in the GO biological process branch), where the entity carries out some part of the process, regulates it, or acts upstream of it.</td>
</tr>
<tr>
<td><a href="MacromolecularMachineToCellularComponentAssociation/">MacromolecularMachineToCellularComponentAssociation</a></td>
<td>A functional association between a macromolecular machine (gene, gene product or complex) and a cellular component (as represented in the GO cellular component branch), where the entity carries out its function in the cellular component.</td>
</tr>
<tr>
<td><a href="MacromolecularMachineToMolecularActivityAssociation/">MacromolecularMachineToMolecularActivityAssociation</a></td>
<td>A functional association between a macromolecular machine (gene, gene product or complex) and a molecular activity (as represented in the GO molecular function branch), where the entity carries out the activity, or contributes to its execution.</td>
</tr>
<tr>
<td><a href="MaterialSampleDerivationAssociation/">MaterialSampleDerivationAssociation</a></td>
<td>An association between a material sample and the material entity from which it is derived.</td>
</tr>
<tr>
<td><a href="MaterialSampleToDiseaseOrPhenotypicFeatureAssociation/">MaterialSampleToDiseaseOrPhenotypicFeatureAssociation</a></td>
<td>An association between a material sample and a disease or phenotype.</td>
</tr>
<tr>
<td><a href="MolecularActivityToChemicalEntityAssociation/">MolecularActivityToChemicalEntityAssociation</a></td>
<td>Added in response to capturing relationship between microbiome activities as measured via measurements of blood analytes as collected via blood and stool samples</td>
</tr>
<tr>
<td><a href="MolecularActivityToMolecularActivityAssociation/">MolecularActivityToMolecularActivityAssociation</a></td>
<td>Added in response to capturing relationship between microbiome activities as measured via measurements of blood analytes as collected via blood and stool samples</td>
</tr>
<tr>
<td><a href="MolecularActivityToPathwayAssociation/">MolecularActivityToPathwayAssociation</a></td>
<td>Association that holds the relationship between a reaction and the pathway it participates in.</td>
</tr>
<tr>
<td><a href="NamedThingAssociatedWithLikelihoodOfNamedThingAssociation/">NamedThingAssociatedWithLikelihoodOfNamedThingAssociation</a></td>
<td></td>
</tr>
<tr>
<td><a href="OrganismTaxonToEnvironmentAssociation/">OrganismTaxonToEnvironmentAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="OrganismTaxonToOrganismTaxonAssociation/">OrganismTaxonToOrganismTaxonAssociation</a></td>
<td>A relationship between two organism taxon nodes</td>
</tr>
<tr>
<td><a href="OrganismTaxonToOrganismTaxonInteraction/">OrganismTaxonToOrganismTaxonInteraction</a></td>
<td>An interaction relationship between two taxa. This may be a symbiotic relationship (encompassing mutualism and parasitism), or it may be non-symbiotic. Example: plague transmitted_by flea; cattle domesticated_by Homo sapiens; plague infects Homo sapiens</td>
</tr>
<tr>
<td><a href="OrganismTaxonToOrganismTaxonSpecialization/">OrganismTaxonToOrganismTaxonSpecialization</a></td>
<td>A child-parent relationship between two taxa. For example: Homo sapiens subclass_of Homo</td>
</tr>
<tr>
<td><a href="OrganismToOrganismAssociation/">OrganismToOrganismAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="OrganismalEntityAsAModelOfDiseaseAssociation/">OrganismalEntityAsAModelOfDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="PairwiseGeneToGeneInteraction/">PairwiseGeneToGeneInteraction</a></td>
<td>An interaction between two genes or two gene products. May be physical (e.g. protein binding) or genetic (between genes). May be symmetric (e.g. protein interaction) or directed (e.g. phosphorylation)</td>
</tr>
<tr>
<td><a href="PairwiseMolecularInteraction/">PairwiseMolecularInteraction</a></td>
<td>An interaction at the molecular level between two physical entities</td>
</tr>
<tr>
<td><a href="PhenotypicFeatureToDiseaseAssociation/">PhenotypicFeatureToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="PhenotypicFeatureToPhenotypicFeatureAssociation/">PhenotypicFeatureToPhenotypicFeatureAssociation</a></td>
<td>Association between two concept nodes of phenotypic character, qualified by the predicate used. This association may typically be used to specify 'similar_to' or 'member_of' relationships.</td>
</tr>
<tr>
<td><a href="PopulationToPopulationAssociation/">PopulationToPopulationAssociation</a></td>
<td>An association between a two populations</td>
</tr>
<tr>
<td><a href="ProcessRegulatesProcessAssociation/">ProcessRegulatesProcessAssociation</a></td>
<td>Describes a regulatory relationship between two genes or gene products.</td>
</tr>
<tr>
<td><a href="ReactionToCatalystAssociation/">ReactionToCatalystAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ReactionToParticipantAssociation/">ReactionToParticipantAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="SequenceAssociation/">SequenceAssociation</a></td>
<td>An association between a sequence feature and a nucleic acid entity it is localized to.</td>
</tr>
<tr>
<td><a href="SequenceFeatureRelationship/">SequenceFeatureRelationship</a></td>
<td>For example, a particular exon is part of a particular transcript or gene</td>
</tr>
<tr>
<td><a href="SequenceVariantModulatesTreatmentAssociation/">SequenceVariantModulatesTreatmentAssociation</a></td>
<td>An association between a sequence variant and a treatment or health intervention. The treatment object itself encompasses both the disease and the drug used.</td>
</tr>
<tr>
<td><a href="TaxonToTaxonAssociation/">TaxonToTaxonAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="TranscriptToGeneRelationship/">TranscriptToGeneRelationship</a></td>
<td>A gene is a collection of transcripts</td>
</tr>
<tr>
<td><a href="VariantAsAModelOfDiseaseAssociation/">VariantAsAModelOfDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="VariantToDiseaseAssociation/">VariantToDiseaseAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="VariantToGeneAssociation/">VariantToGeneAssociation</a></td>
<td>An association between a variant and a gene, where the variant has a genetic association with the gene (i.e. is in linkage disequilibrium)</td>
</tr>
<tr>
<td><a href="VariantToGeneExpressionAssociation/">VariantToGeneExpressionAssociation</a></td>
<td>An association between a variant and expression of a gene (i.e. e-QTL)</td>
</tr>
<tr>
<td><a href="VariantToPhenotypicFeatureAssociation/">VariantToPhenotypicFeatureAssociation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="VariantToPopulationAssociation/">VariantToPopulationAssociation</a></td>
<td>An association between a variant and a population, where the variant has particular frequency in the population</td>
</tr>
</tbody>
</table>
<h2 id="association-properties">Association Properties</h2>
<table>
<thead>
<tr>
<th>Property</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="adjusted_p_value/">adjusted_p_value</a></td>
<td>The adjusted p-value is the probability of obtaining test results at least as extreme as the results actually observed, under the assumption that the null hypothesis is correct, adjusted for multiple comparisons. P is always italicized and capitalized. The actual P value* should be expressed (P=. 04) rather than expressing a statement of inequality (P&lt;. 05), unless P&lt;.</td>
</tr>
<tr>
<td><a href="affinity/">affinity</a></td>
<td>The numerical value describing the strength of an affinity between two entities.  For instance, if a chemical inhibits a protein with a pIC50 of 8.6, the affinity is 8.6. Used in conjunction with the affinity parameter slot.</td>
</tr>
<tr>
<td><a href="affinity_parameter/">affinity_parameter</a></td>
<td>The type of parameter describing the strength of an affinity between two entities.  For instance, if a chemical inhibits a protein with a pIC50 of 8.6, the affinity parameter is pIC50.  Used in conjunction with the affinity slot.</td>
</tr>
<tr>
<td><a href="agent_type/">agent_type</a></td>
<td>Describes the high-level category of agent who originally generated a statement of knowledge or other type of information.</td>
</tr>
<tr>
<td><a href="aggregator_knowledge_source/">aggregator_knowledge_source</a></td>
<td>An intermediate aggregator resource from which knowledge expressed in an Association was retrieved downstream of the original source, on its path to its current serialized form.</td>
</tr>
<tr>
<td><a href="allelic_requirement/">allelic_requirement</a></td>
<td>The allele configuration of a particular gene or variant required for the expression of a disease or phenotype in a specific patient or instance.</td>
</tr>
<tr>
<td><a href="anatomical_context_qualifier/">anatomical_context_qualifier</a></td>
<td>A statement qualifier representing an anatomical location where an relationship expressed in an association took place (can be a tissue, cell type, or sub-cellular location).</td>
</tr>
<tr>
<td><a href="aspect_qualifier/">aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.</td>
</tr>
<tr>
<td><a href="associated_environmental_context/">associated_environmental_context</a></td>
<td>An attribute that can be applied to an association where the association holds between two entities located or occurring in a particular environment. For example, two microbial taxa may interact in the context of a human gut; a disease may give rise to a particular phenotype in a particular environmental exposure.</td>
</tr>
<tr>
<td># TODO: add examples of values for this property.</td>
<td></td>
</tr>
<tr>
<td><a href="association_slot/">association_slot</a></td>
<td>any slot that relates an association to another entity</td>
</tr>
<tr>
<td><a href="base_coordinate/">base_coordinate</a></td>
<td>A position in the base coordinate system.  Base coordinates start at position 1 instead of position 0.</td>
</tr>
<tr>
<td><a href="bonferonni_adjusted_p_value/">bonferonni_adjusted_p_value</a></td>
<td>The Bonferroni correction is an adjustment made to P values when several dependent or independent statistical tests are being performed simultaneously on a single data set. To perform a Bonferroni correction, divide the critical P value (α) by the number of comparisons being made.  P is always italicized and capitalized. The actual P value* should be expressed (P=. 04) rather than expressing a statement of inequality (P&lt;. 05), unless P&lt;.</td>
</tr>
<tr>
<td><a href="catalyst_qualifier/">catalyst_qualifier</a></td>
<td>a qualifier that connects an association between two causally connected entities (for example, two chemical entities, or a chemical entity in that changes location) and the gene product, gene, or complex that enables or catalyzes the change.</td>
</tr>
<tr>
<td><a href="causal_mechanism_qualifier/">causal_mechanism_qualifier</a></td>
<td>A statement qualifier representing a type of molecular control mechanism through which an effect of a chemical on a gene or gene product is mediated (e.g. 'agonism', 'inhibition', 'allosteric modulation', 'channel blocker')</td>
</tr>
<tr>
<td><a href="chi_squared_dof/">chi_squared_dof</a></td>
<td>Degrees of freedom (dof) in a chi-squared test referring to the number of values in the final calculation of a statistic that are free to vary</td>
</tr>
<tr>
<td><a href="chi_squared_p/">chi_squared_p</a></td>
<td>The chi-square p-value tells you the probability that the observed differences (or associations) in your data occurred by random chance, assuming the null hypothesis is true.</td>
</tr>
<tr>
<td><a href="chi_squared_statistic/">chi_squared_statistic</a></td>
<td>The chi-squared statistic measures how much observed data deviate from expected values under the null hypothesis.</td>
</tr>
<tr>
<td><a href="clinical_approval_status/">clinical_approval_status</a></td>
<td></td>
</tr>
<tr>
<td><a href="clinical_modifier_qualifier/">clinical_modifier_qualifier</a></td>
<td>the method or process of administering a pharmaceutical compound to achieve a therapeutic effect in humans or animals.</td>
</tr>
<tr>
<td><a href="concept_count_object/">concept_count_object</a></td>
<td>The number of instances in a dataset/cohort whose records contain the concept in the object slot of an association.</td>
</tr>
<tr>
<td><a href="concept_count_subject/">concept_count_subject</a></td>
<td>The number of instances in a dataset/cohort whose records contain the concept in the subject slot of an association.</td>
</tr>
<tr>
<td><a href="concept_pair_count/">concept_pair_count</a></td>
<td>The number of instances in a dataset/cohort whose records contain both the subject and object concept of an association.</td>
</tr>
<tr>
<td><a href="context_qualifier/">context_qualifier</a></td>
<td>Restricts the setting/context/location where the core concept (or qualified core concept) resides or occurs.</td>
</tr>
<tr>
<td><a href="dataset_count/">dataset_count</a></td>
<td>The total number of instances (e.g., number of patients, number of rows, etc) in a dataset/cohort.</td>
</tr>
<tr>
<td><a href="derivative_qualifier/">derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.</td>
</tr>
<tr>
<td><a href="direction_qualifier/">direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree.</td>
</tr>
<tr>
<td><a href="disease_context_qualifier/">disease_context_qualifier</a></td>
<td>A context qualifier representing a disease or condition in which a relationship expressed in an association took place.</td>
</tr>
<tr>
<td><a href="end_coordinate/">end_coordinate</a></td>
<td>The position at which the subject genomic entity ends on the chromosome or other entity to which it is located on.</td>
</tr>
<tr>
<td><a href="end_interbase_coordinate/">end_interbase_coordinate</a></td>
<td>The position at which the subject nucleic acid entity ends on the chromosome or other entity to which it is located on.</td>
</tr>
<tr>
<td><a href="evidence_count/">evidence_count</a></td>
<td>The number of evidence instances that are connected to an association.</td>
</tr>
<tr>
<td><a href="expected_count/">expected_count</a></td>
<td>The expected (calculated) number of instances in a dataset/cohort whose records contain both the subject and object concept of an association if the subject and object concepts are independent.</td>
</tr>
<tr>
<td><a href="expression_site/">expression_site</a></td>
<td>location in which gene or protein expression takes place. May be cell, tissue, or organ.</td>
</tr>
<tr>
<td><a href="extraction_confidence_score/">extraction_confidence_score</a></td>
<td>A quantitative confidence value that represents the probability of obtaining a result at least as extreme as that actually obtained, assuming that the actual value was the result of chance alone.</td>
</tr>
<tr>
<td><a href="FDA_adverse_event_level/">FDA_adverse_event_level</a></td>
<td></td>
</tr>
<tr>
<td><a href="fisher_exact_odds_ratio/">fisher_exact_odds_ratio</a></td>
<td>The Fisher Exact Test is used to determine whether there is a non-random association between two categorical variables in a 2×2 contingency table, especially when sample sizes are small. The odds ratio (OR) quantifies the strength of that association.</td>
</tr>
<tr>
<td>OR = 1 implies No association</td>
<td></td>
</tr>
<tr>
<td>OR &gt; 1 implies Positive association (Group A more likely to have Outcome 1)</td>
<td></td>
</tr>
<tr>
<td>OR &lt; 1 implies Negative association (Group A less likely to have Outcome 1)</td>
<td></td>
</tr>
<tr>
<td><a href="fisher_exact_p/">fisher_exact_p</a></td>
<td>The Fisher exact p-value tells you the probability of observing a table as extreme as (or more extreme than) your actual data, assuming that the null hypothesis of independence is true. It's most commonly used for 2×2 contingency tables, especially when sample sizes are small or expected counts are low.</td>
</tr>
<tr>
<td><a href="form_or_variant_qualifier/">form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.</td>
</tr>
<tr>
<td><a href="frequency_qualifier/">frequency_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state how frequent the phenotype is observed in the subject</td>
</tr>
<tr>
<td><a href="genome_build/">genome_build</a></td>
<td>The version of the genome on which a feature is located. For example, GRCh38 for Homo sapiens.</td>
</tr>
<tr>
<td><a href="has_confidence_level/">has_confidence_level</a></td>
<td>connects an association to a qualitative term denoting the level of confidence</td>
</tr>
<tr>
<td><a href="has_confidence_score/">has_confidence_score</a></td>
<td>connects an association to a quantitative (numeric) value that can be interpreted as an indicator of the degree of confidence that a piece of information is true, and accurately reflects the aspect of reality it is about.</td>
</tr>
<tr>
<td><a href="has_evidence/">has_evidence</a></td>
<td>connects an association to an instance of supporting evidence</td>
</tr>
<tr>
<td><a href="has_supporting_studies/">has_supporting_studies</a></td>
<td>A study that produced information used as evidence to generate the knowledge expressed in an Association.</td>
</tr>
<tr>
<td><a href="has_supporting_study_result/">has_supporting_study_result</a></td>
<td>connects an association to an instance of supporting study result</td>
</tr>
<tr>
<td><a href="interacting_molecules_category/">interacting_molecules_category</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="interbase_coordinate/">interbase_coordinate</a></td>
<td>A position in interbase coordinates. Interbase coordinates start at position 0 instead of position 1. This is applied to a sequence localization edge.</td>
</tr>
<tr>
<td><a href="knowledge_level/">knowledge_level</a></td>
<td>Describes the level of knowledge expressed in a statement, based on the reasoning or analysis methods used to generate the statement, or the scope or specificity of what the statement expresses to be true.</td>
</tr>
<tr>
<td><a href="knowledge_source/">knowledge_source</a></td>
<td>An Information Resource from which the knowledge expressed in an Association was retrieved, directly or indirectly. This can be any resource through which the knowledge passed on its way to its currently serialized form. In practice, implementers should use one of the more specific subtypes of this generic property.</td>
</tr>
<tr>
<td><a href="ln_ratio/">ln_ratio</a></td>
<td>the natural log of the ratio of co-occurrence to expected</td>
</tr>
<tr>
<td><a href="ln_ratio_confidence_interval/">ln_ratio_confidence_interval</a></td>
<td>The 99% confidence interval for the ln_ratio calculation (i.e. the range of values within which the true value has a 99% chance of falling)</td>
</tr>
<tr>
<td><a href="log_odds_ratio/">log_odds_ratio</a></td>
<td>The natural logarithm of the odds ratio (OR), or the ratio of the odds of an event Y occurring in an exposed group versus the odds of an event Y occurring in a non-exposed group.</td>
</tr>
<tr>
<td><a href="log_odds_ratio_95_ci/">log_odds_ratio_95_ci</a></td>
<td>The ninety-five percent confidence range in which the true log odds ratio for the sample population falls. To calculate the 95% confidence interval (CI) for a log odds ratio (a pair of numbers), you need the standard error (SE) of the log odds ratio.  This interval helps you understand the precision of your estimate and whether the association is statistically significant.</td>
</tr>
<tr>
<td><a href="logical_interpretation/">logical_interpretation</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="max_research_phase/">max_research_phase</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="mechanism_of_action/">mechanism_of_action</a></td>
<td>a boolean flag to indicate if the edge is part of a path or subgraph of a knowledge graph that constitutes the mechanism of action for a result.</td>
</tr>
<tr>
<td><a href="negated/">negated</a></td>
<td>if set to true, then the association is negated i.e. is not true</td>
</tr>
<tr>
<td><a href="object/">object</a></td>
<td>connects an association to the object of the association. For example, in a gene-to-phenotype association, the gene is subject and phenotype is object.</td>
</tr>
<tr>
<td><a href="object_aspect_qualifier/">object_aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.  This qualifier specifies a change in the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="object_category/">object_category</a></td>
<td>Used to hold the biolink class/category of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="object_category_closure/">object_category_closure</a></td>
<td>Used to hold the object category closure of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="object_closure/">object_closure</a></td>
<td>Used to hold the object closure of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="object_context_qualifier/">object_context_qualifier</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="object_derivative_qualifier/">object_derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.  This qualifier is for the object of an association (or statement).</td>
</tr>
<tr>
<td><a href="object_direction_qualifier/">object_direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree. This qualifier qualifies the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="object_form_or_variant_qualifier/">object_form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.  This qualifier specifies a change in the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="object_label_closure/">object_label_closure</a></td>
<td>Used to hold the object label closure of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="object_location_in_text/">object_location_in_text</a></td>
<td>Character offsets for the text span(s) in the supporting text corresponding to the object concept of the extracted assertion</td>
</tr>
<tr>
<td><a href="object_namespace/">object_namespace</a></td>
<td>Used to hold the object namespace of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="object_part_qualifier/">object_part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).  This qualifier is for the object of an association (or statement).</td>
</tr>
<tr>
<td><a href="object_specialization_qualifier/">object_specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="onset_qualifier/">onset_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state when the phenotype appears is in the subject.</td>
</tr>
<tr>
<td><a href="original_object/">original_object</a></td>
<td>used to hold the original object of a relation (or predicate) that an external knowledge source uses before transformation to match the biolink-model specification.</td>
</tr>
<tr>
<td><a href="original_predicate/">original_predicate</a></td>
<td>used to hold the original relation/predicate that an external knowledge source uses before transformation to match the biolink-model specification.</td>
</tr>
<tr>
<td><a href="original_subject/">original_subject</a></td>
<td>used to hold the original subject of a relation (or predicate) that an external knowledge source uses before transformation to match the biolink-model specification.</td>
</tr>
<tr>
<td><a href="p_value/">p_value</a></td>
<td>A quantitative confidence value that represents the probability of obtaining a result at least as extreme as that actually obtained, assuming that the actual value was the result of chance alone.</td>
</tr>
<tr>
<td><a href="part_qualifier/">part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).</td>
</tr>
<tr>
<td><a href="phase/">phase</a></td>
<td>The phase for a coding sequence entity. For example, phase of a CDS as represented in a GFF3 with a value of 0, 1 or 2.</td>
</tr>
<tr>
<td><a href="phenotypic_state/">phenotypic_state</a></td>
<td>in experiments (e.g. gene expression) assaying diseased or unhealthy tissue, the phenotypic state can be put here, e.g. MONDO ID. For healthy tissues, use XXX.</td>
</tr>
<tr>
<td><a href="population_context_qualifier/">population_context_qualifier</a></td>
<td>a biological population (general, study, cohort, etc.) with a specific set of characteristics to constrain an association.</td>
</tr>
<tr>
<td><a href="predicate/">predicate</a></td>
<td>A high-level grouping for the relationship type. AKA minimal predicate. This is analogous to category for nodes.</td>
</tr>
<tr>
<td><a href="primary_knowledge_source/">primary_knowledge_source</a></td>
<td>The most upstream source of the knowledge expressed in an Association that an implementer can identify.  Performing a rigorous analysis of upstream data providers is expected; every effort is made to catalog the most upstream source of data in this property.  Only one data source should be declared primary in any association.  "aggregator knowledge source" can be used to capture non-primary sources.</td>
</tr>
<tr>
<td><a href="publications/">publications</a></td>
<td>One or more publications that report the statement expressed in an Association, or provide information used as evidence supporting this statement.</td>
</tr>
<tr>
<td><a href="qualified_predicate/">qualified_predicate</a></td>
<td>Predicate to be used in an association when subject and object qualifiers are present and the full reading of the statement requires a qualification to the predicate in use in order to refine or increase the specificity of the full statement reading.  This qualifier holds a relationship to be used instead of that expressed by the primary predicate, in a ‘full statement’ reading of the association, where qualifier-based semantics are included.  This is necessary only in cases where the primary predicate does not work in a full statement reading.</td>
</tr>
<tr>
<td><a href="qualifier/">qualifier</a></td>
<td>grouping slot for all qualifiers on an edge.  useful for testing compliance with association classes</td>
</tr>
<tr>
<td><a href="qualifiers/">qualifiers</a></td>
<td>connects an association to qualifiers that modify or qualify the meaning of that association</td>
</tr>
<tr>
<td><a href="quantifier_qualifier/">quantifier_qualifier</a></td>
<td>A measurable quantity for the object of the association</td>
</tr>
<tr>
<td><a href="reaction_balanced/">reaction_balanced</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="reaction_direction/">reaction_direction</a></td>
<td>the direction of a reaction as constrained by the direction enum (ie: left_to_right, neutral, etc.)</td>
</tr>
<tr>
<td><a href="reaction_side/">reaction_side</a></td>
<td>the side of a reaction being modeled (ie: left or right)</td>
</tr>
<tr>
<td><a href="relative_frequency_object/">relative_frequency_object</a></td>
<td>The frequency at which subject and object concepts co-occur in records within a dataset/cohort, relative to the frequency at which the object concept appears in these same records.</td>
</tr>
<tr>
<td><a href="relative_frequency_object_confidence_interval/">relative_frequency_object_confidence_interval</a></td>
<td>The 99% confidence interval for the relative_frequency_object calculation (i.e. the range of values within which the true value has a 99% chance of falling)</td>
</tr>
<tr>
<td><a href="relative_frequency_subject/">relative_frequency_subject</a></td>
<td>The frequency at which subject and object concepts co-occur in records within a dataset/cohort, relative to the frequency at which the subject concept appears in these same records.</td>
</tr>
<tr>
<td><a href="relative_frequency_subject_confidence_interval/">relative_frequency_subject_confidence_interval</a></td>
<td>The 99% confidence interval for the relative_frequency_subject calculation (i.e. the range of values within which the true value has a 99% chance of falling)</td>
</tr>
<tr>
<td><a href="response_context_qualifier/">response_context_qualifier</a></td>
<td>a biological response (general, study, cohort, etc.) with a specific set of characteristics to constrain an association.</td>
</tr>
<tr>
<td><a href="response_target_context_qualifier/">response_target_context_qualifier</a></td>
<td>a biological response target (a patient, a cohort, a model system, a cell line, a sample of biological material, etc.)</td>
</tr>
<tr>
<td><a href="semmed_agreement_count/">semmed_agreement_count</a></td>
<td>The number of times this concept has been asserted in the SemMedDB literature database.</td>
</tr>
<tr>
<td><a href="sequence_localization_attribute/">sequence_localization_attribute</a></td>
<td>An attribute that can be applied to a genome sequence localization edge. These edges connect a nucleic acid entity such as an exon to an entity such as a chromosome. Edge properties are used to ascribe specific positional information and other metadata to the localization. In pragmatic terms this can be thought of as columns in a GFF3 line.</td>
</tr>
<tr>
<td><a href="sequence_variant_qualifier/">sequence_variant_qualifier</a></td>
<td>a qualifier used in an association with the variant</td>
</tr>
<tr>
<td><a href="severity_qualifier/">severity_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state how severe the phenotype is in the subject</td>
</tr>
<tr>
<td><a href="sex_qualifier/">sex_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state whether the association is specific to a particular sex.</td>
</tr>
<tr>
<td><a href="sources/">sources</a></td>
<td>A set of RetrievalSources, which traces where the statement expressed in an Association came from. For example, the provenance of a Gene-Chemical Edge might be traced through the Translator Resource that provided it (e.g. MolePro) to one or more intermediate aggregator resources (e.g. ChEMBL), and finally to the resource that originally created/curated it (e.g. ClinicalTrials.org).</td>
</tr>
<tr>
<td><a href="specialization_qualifier/">specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="species_context_qualifier/">species_context_qualifier</a></td>
<td>A statement qualifier representing a taxonomic category of species in which a relationship expressed in an association took place.</td>
</tr>
<tr>
<td><a href="stage_qualifier/">stage_qualifier</a></td>
<td>stage during which gene or protein expression of takes place.</td>
</tr>
<tr>
<td><a href="start_coordinate/">start_coordinate</a></td>
<td>The position at which the subject genomic entity starts on the chromosome or other entity to which it is located on. (ie: the start of the sequence being referenced is 1).</td>
</tr>
<tr>
<td><a href="start_interbase_coordinate/">start_interbase_coordinate</a></td>
<td>The position at which the subject nucleic acid entity starts on the chromosome or other entity to which it is located on. (ie: the start of the sequence being referenced is 0).</td>
</tr>
<tr>
<td><a href="statement_qualifier/">statement_qualifier</a></td>
<td>A property that qualifies the entirety of the statement made in an association.  It applies to both a fully qualified subject and a fully qualified object as well as the predicate and qualified predicate in an association.</td>
</tr>
<tr>
<td><a href="stoichiometry/">stoichiometry</a></td>
<td>the relationship between the relative quantities of substances taking part in a reaction or forming a compound, typically a ratio of whole integers.</td>
</tr>
<tr>
<td><a href="strand/">strand</a></td>
<td>The strand on which a feature is located. Has a value of '+' (sense strand or forward strand) or '-' (anti-sense strand or reverse strand).</td>
</tr>
<tr>
<td><a href="subject/">subject</a></td>
<td>connects an association to the subject of the association. For example, in a gene-to-phenotype association, the gene is subject and phenotype is object.</td>
</tr>
<tr>
<td><a href="subject_aspect_qualifier/">subject_aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.  This qualifier specifies a change in the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="subject_category/">subject_category</a></td>
<td>Used to hold the biolink class/category of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="subject_category_closure/">subject_category_closure</a></td>
<td>Used to hold the subject category closure of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="subject_closure/">subject_closure</a></td>
<td>Used to hold the subject closure of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="subject_context_qualifier/">subject_context_qualifier</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="subject_derivative_qualifier/">subject_derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.  This qualifier is for the subject of an association (or statement).</td>
</tr>
<tr>
<td><a href="subject_direction_qualifier/">subject_direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree. This qualifier qualifies the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="subject_form_or_variant_qualifier/">subject_form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.  This qualifier specifies a change in the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="subject_label_closure/">subject_label_closure</a></td>
<td>Used to hold the subject label closure of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="subject_location_in_text/">subject_location_in_text</a></td>
<td>Character offsets for the text span(s) in the supporting text corresponding to the subject concept of the extracted assertion.</td>
</tr>
<tr>
<td><a href="subject_namespace/">subject_namespace</a></td>
<td>Used to hold the subject namespace of an association. This is a denormalized field used primarily in the SQL serialization of a knowledge graph via KGX.</td>
</tr>
<tr>
<td><a href="subject_part_qualifier/">subject_part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).  This qualifier is for the subject of an association (or statement).</td>
</tr>
<tr>
<td><a href="subject_specialization_qualifier/">subject_specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="support_graphs/">support_graphs</a></td>
<td>A list of knowledge graphs that support the existence of this association.</td>
</tr>
<tr>
<td><a href="supporting_data_set/">supporting_data_set</a></td>
<td>A set of data used as evidence to generate the knowledge expressed in an Association (e.g. through computation on, reasoning or inference over the retrieved data).</td>
</tr>
<tr>
<td><a href="supporting_data_source/">supporting_data_source</a></td>
<td>An Information Resource from which data was retrieved and subsequently used as evidence to generate the knowledge expressed in an Association (e.g. through computation on, reasoning or inference over the retrieved data).</td>
</tr>
<tr>
<td><a href="supporting_document_type/">supporting_document_type</a></td>
<td>The document type (e.g., Journal Article, Case Study, Preprint) for the supporting document used in a Text Mining Result.</td>
</tr>
<tr>
<td><a href="supporting_document_year/">supporting_document_year</a></td>
<td>The document year (typically the publication year) for the supporting document used in a Text Mining Result.</td>
</tr>
<tr>
<td><a href="supporting_documents/">supporting_documents</a></td>
<td>One or more referenceable documents that report the statement expressed in an Association, or provide information used as evidence supporting this statement.</td>
</tr>
<tr>
<td><a href="supporting_study_cohort/">supporting_study_cohort</a></td>
<td>A description of a study population/cohort that was interrogated to provide evidence for the association (e.g. the inclusion and exclusion criteria).</td>
</tr>
<tr>
<td><a href="supporting_study_context/">supporting_study_context</a></td>
<td>A term or terms describing the experimental setting/context in which evidence supporting the Association was generated ('context' may be defined by many factors, including taxon, model system (e.g. cell line type), tissue type, disease, etc.).</td>
</tr>
<tr>
<td><a href="supporting_study_date_range/">supporting_study_date_range</a></td>
<td>The date range over which data was collected in a study that provided evidence for an Association.</td>
</tr>
<tr>
<td><a href="supporting_study_metadata/">supporting_study_metadata</a></td>
<td>Information about a study used to generate information used as evidence to support the knowledge expressed in an Association. In practice, data creators should use one of the more specific subtypes of this property.</td>
</tr>
<tr>
<td><a href="supporting_study_method_description/">supporting_study_method_description</a></td>
<td>A uri or curie pointing to information about the methodology used to generate data supporting an Association.</td>
</tr>
<tr>
<td><a href="supporting_study_method_type/">supporting_study_method_type</a></td>
<td>A type of method that was applied in a study used to generate the information used as evidence (e.g. a type of experimental assay, or statistical calculation, or computational analysis).</td>
</tr>
<tr>
<td><a href="supporting_study_size/">supporting_study_size</a></td>
<td>The sample size used in a study that provided evidence for the association (e.g. 'n' of a cohort for a clinical study).</td>
</tr>
<tr>
<td><a href="supporting_text/">supporting_text</a></td>
<td>The segment of text from a document that supports the mined assertion.</td>
</tr>
<tr>
<td><a href="supporting_text_section_type/">supporting_text_section_type</a></td>
<td>The section of the supporting text of a Text Mining Result within the supporting document. This is in the form of the name of the document section (e.g., Abstract, Introduction) that contains the supporting text.</td>
</tr>
<tr>
<td><a href="temporal_context_qualifier/">temporal_context_qualifier</a></td>
<td>a constraint of time placed upon the truth value of an association. for time intervales, use temporal interval qualifier.</td>
</tr>
<tr>
<td><a href="temporal_interval_qualifier/">temporal_interval_qualifier</a></td>
<td>a constraint of a time interval placed upon the truth value of an association.</td>
</tr>
<tr>
<td><a href="total_sample_size/">total_sample_size</a></td>
<td>The total number of patients or participants within a sample population.</td>
</tr>
<tr>
<td><a href="z_score/">z_score</a></td>
<td>A measure of the divergence of an individual experimental result from the most probable result, the mean. Z is expressed in terms of the number of standard deviations from the mean value.</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AffinityParameterEnum/">AffinityParameterEnum</a></td>
<td>The types of parameters that can be used to describe the affinity between two entities, characteristically chemicals and proteins.</td>
</tr>
<tr>
<td><a href="AgentTypeEnum/">AgentTypeEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ApprovalStatusEnum/">ApprovalStatusEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="CausalMechanismQualifierEnum/">CausalMechanismQualifierEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ChemicalEntityDerivativeEnum/">ChemicalEntityDerivativeEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ChemicalOrGeneOrGeneProductFormOrVariantEnum/">ChemicalOrGeneOrGeneProductFormOrVariantEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ClinicalApprovalStatusEnum/">ClinicalApprovalStatusEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ClinicalTrialStatusEnum/">ClinicalTrialStatusEnum</a></td>
<td>Enumeration of clinical trial statuses indicating the recruitment state, availability, or regulatory status of a clinical study or intervention.</td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td><a href="DirectionQualifierEnum/">DirectionQualifierEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="DrugAvailabilityEnum/">DrugAvailabilityEnum</a></td>
<td></td>
</tr>
<tr>
<td><a href="DrugDeliveryEnum/">DrugDeliveryEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="DruggableGeneCategoryEnum/">DruggableGeneCategoryEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="FDAIDAAdverseEventEnum/">FDAIDAAdverseEventEnum</a></td>
<td>please consult with the FDA guidelines as proposed in this document: https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfcfr/cfrsearch.cfm?fr=312.32</td>
</tr>
<tr>
<td><a href="GeneOrGeneProductOrChemicalEntityAspectEnum/">GeneOrGeneProductOrChemicalEntityAspectEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="GeneOrGeneProductOrChemicalPartQualifierEnum/">GeneOrGeneProductOrChemicalPartQualifierEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="KnowledgeLevelEnum/">KnowledgeLevelEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="LogicalInterpretationEnum/">LogicalInterpretationEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="PhaseEnum/">PhaseEnum</a></td>
<td>phase</td>
</tr>
<tr>
<td><a href="ReactionDirectionEnum/">ReactionDirectionEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ReactionSideEnum/">ReactionSideEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ResearchPhaseEnum/">ResearchPhaseEnum</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="ResourceRoleEnum/">ResourceRoleEnum</a></td>
<td>The role played by the information reource in serving as a source for an edge in a TRAPI message. Note that a given Edge should have one and only one 'primary' source, and may have any number of 'aggregator' or 'supporting data' sources.  This enumeration is found in Biolink Model, but is repeated here for convenience.</td>
</tr>
<tr>
<td><a href="ResponseEnum/">ResponseEnum</a></td>
<td>A response to a treatment or intervention</td>
</tr>
<tr>
<td><a href="ResponseTargetEnum/">ResponseTargetEnum</a></td>
<td>The target of a treatment or intervention</td>
</tr>
<tr>
<td><a href="SequenceEnum/">SequenceEnum</a></td>
<td>type of sequence</td>
</tr>
<tr>
<td><a href="StrandEnum/">StrandEnum</a></td>
<td>strand</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="ModelOrganismDatabase/">ModelOrganismDatabase</a></td>
<td>Subset that is relevant for a typical Model Organism Database (MOD)</td>
</tr>
<tr>
<td><a href="Samples/">Samples</a></td>
<td>Sample/biosample datamodel</td>
</tr>
<tr>
<td><a href="Testing/">Testing</a></td>
<td>TBD</td>
</tr>
<tr>
<td><a href="TranslatorMinimal/">TranslatorMinimal</a></td>
<td>Minimum subset of translator work</td>
</tr>
</tbody>
</table>
<h4 id="citing-biolink-model">Citing Biolink Model</h4>
<p><small> Unni DR, Moxon SAT, Bada M, Brush M, Bruskiewich R, Caufield JH, Clemons PA, Dancik V, Dumontier M, Fecho K, Glusman G,
Hadlock JJ, Harris NL, Joshi A, Putman T, Qin G, Ramsey SA, Shefchek KA, Solbrig H, Soman K, Thessen AE, Haendel MA,
Bizon C, Mungall CJ, The Biomedical Data Translator Consortium (2022).
Biolink Model: A universal schema for knowledge graphs in clinical, biomedical, and translational science. Clin Transl Sci. Wiley; 2022 Jun 6; <a href="https://onlinelibrary.wiley.com/doi/10.1111/cts.13302">https://onlinelibrary.wiley.com/doi/10.1111/cts.13302</a>
</small></p>









  




                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": ".", "features": ["content.tabs.link"], "search": "assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="assets/javascripts/bundle.56ea9cef.min.js"></script>
      
    
  </body>
</html>