
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://biolink.github.io/biolink-model/TranslatorMinimal/">
      
      
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.6.15">
    
    
      
        <title>Subset: TranslatorMinimal - Biolink Model Documentation</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.342714a4.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../style.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      
  


  
  

<script id="__analytics">function __md_analytics(){function e(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],e("js",new Date),e("config","G-J23SCD6JB8"),document.addEventListener("DOMContentLoaded",(function(){document.forms.search&&document.forms.search.query.addEventListener("blur",(function(){this.value&&e("event","search",{search_term:this.value})}));document$.subscribe((function(){var t=document.forms.feedback;if(void 0!==t)for(var a of t.querySelectorAll("[type=submit]"))a.addEventListener("click",(function(a){a.preventDefault();var n=document.location.pathname,d=this.getAttribute("data-md-value");e("event","feedback",{page:n,data:d}),t.firstElementChild.disabled=!0;var r=t.querySelector(".md-feedback__note [data-md-value='"+d+"']");r&&(r.hidden=!1)})),t.hidden=!1})),location$.subscribe((function(t){e("config","G-J23SCD6JB8",{page_path:t.pathname})}))}));var t=document.createElement("script");t.async=!0,t.src="https://www.googletagmanager.com/gtag/js?id=G-J23SCD6JB8",document.getElementById("__analytics").insertAdjacentElement("afterEnd",t)}</script>
  
    <script>"undefined"!=typeof __md_analytics&&__md_analytics()</script>
  

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="black" data-md-color-primary="black" data-md-color-accent="indigo">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#subset-translatorminimal" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Biolink Model Documentation" class="md-header__button md-logo" aria-label="Biolink Model Documentation" data-md-component="logo">
      
  <img src="../images/biolink-logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Biolink Model Documentation
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Subset: TranslatorMinimal
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/biolink/biolink-model" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Biolink Model Documentation" class="md-nav__button md-logo" aria-label="Biolink Model Documentation" data-md-component="logo">
      
  <img src="../images/biolink-logo.png" alt="logo">

    </a>
    Biolink Model Documentation
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/biolink/biolink-model" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.7.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2024 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Overview
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../using-the-modeling-language/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Using the LinkML Modeling Language
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../association-examples-with-qualifiers/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Associations with Qualifiers
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../reading-a-qualifier-based-statement/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Interpreting a Fully Qualified Edge
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../curating-the-model/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Curating Biolink
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../maintaining-the-model/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Maintaining Biolink
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../working-with-the-model/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Working with Biolink
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="https://biolink.github.io/information-resource-registry" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Infores Identifier Registry
    
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_9" >
        
          
          <label class="md-nav__link" for="__nav_9" id="__nav_9_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    Visualizations
    
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_9_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_9">
            <span class="md-nav__icon md-icon"></span>
            Visualizations
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../categories.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Classes (categories)
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../predicates.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Predicates
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../qualifiers.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Qualifiers
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../associations.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Associations
    
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../aspects.html" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    Aspects
    
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../faq/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    FAQ
    
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="subset-translatorminimal">Subset: TranslatorMinimal</h1>
<p><em>Minimum subset of translator work</em></p>
<h1 id="classes-in-subset">Classes in subset:</h1>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../ChemicalEntity/">ChemicalEntity</a></td>
<td>A chemical entity is a physical entity that pertains to chemistry or biochemistry.</td>
</tr>
<tr>
<td><a href="../ChemicalMixture/">ChemicalMixture</a></td>
<td>A chemical mixture is a chemical entity composed of two or more molecular entities.</td>
</tr>
<tr>
<td><a href="../ComplexMolecularMixture/">ComplexMolecularMixture</a></td>
<td>A complex molecular mixture is a chemical mixture composed of two or more molecular entities with unknown concentration and stoichiometry.</td>
</tr>
<tr>
<td><a href="../Disease/">Disease</a></td>
<td>A disorder of structure or function, especially one that produces specific signs, phenotypes or symptoms or that affects a specific location and is not simply a direct result of physical injury.  A disposition to undergo pathological processes that exists in an organism because of one or more disorders in that organism.</td>
</tr>
<tr>
<td><a href="../EpigenomicEntity/">EpigenomicEntity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../Gene/">Gene</a></td>
<td>A region (or regions) that includes all of the sequence elements necessary to encode a functional transcript. A gene locus may include regulatory regions, transcribed regions and/or other functional sequence regions.</td>
</tr>
<tr>
<td><a href="../GenomicEntity/">GenomicEntity</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../MolecularEntity/">MolecularEntity</a></td>
<td>A molecular entity is a chemical entity composed of individual or covalently bonded atoms.</td>
</tr>
<tr>
<td><a href="../MolecularMixture/">MolecularMixture</a></td>
<td>A molecular mixture is a chemical mixture composed of two or more molecular entities with known concentration and stoichiometry.</td>
</tr>
<tr>
<td><a href="../NucleicAcidEntity/">NucleicAcidEntity</a></td>
<td>A nucleic acid entity is a molecular entity characterized by availability in gene databases of nucleotide-based sequence representations of its precise sequence; for convenience of representation, partial sequences of various kinds are included.</td>
</tr>
<tr>
<td><a href="../SmallMolecule/">SmallMolecule</a></td>
<td>A small molecule entity is a molecular entity characterized by availability in small-molecule databases of SMILES, InChI, IUPAC, or other unambiguous representation of its precise chemical structure; for convenience of representation, any valid chemical representation is included, even if it is not strictly molecular (e.g., sodium ion).</td>
</tr>
</tbody>
</table>
<h3 id="chemicalentity">ChemicalEntity</h3>
<p>A chemical entity is a physical entity that pertains to chemistry or biochemistry.</p>
<h3 id="chemicalmixture">ChemicalMixture</h3>
<p>A chemical mixture is a chemical entity composed of two or more molecular entities.</p>
<h3 id="complexmolecularmixture">ComplexMolecularMixture</h3>
<p>A complex molecular mixture is a chemical mixture composed of two or more molecular entities with unknown concentration and stoichiometry.</p>
<h3 id="disease">Disease</h3>
<p>A disorder of structure or function, especially one that produces specific signs, phenotypes or symptoms or that affects a specific location and is not simply a direct result of physical injury.  A disposition to undergo pathological processes that exists in an organism because of one or more disorders in that organism.</p>
<h3 id="epigenomicentity">EpigenomicEntity</h3>
<p>None</p>
<h3 id="gene">Gene</h3>
<p>A region (or regions) that includes all of the sequence elements necessary to encode a functional transcript. A gene locus may include regulatory regions, transcribed regions and/or other functional sequence regions.</p>
<h3 id="genomicentity">GenomicEntity</h3>
<p>None</p>
<h3 id="molecularentity">MolecularEntity</h3>
<p>A molecular entity is a chemical entity composed of individual or covalently bonded atoms.</p>
<h3 id="molecularmixture">MolecularMixture</h3>
<p>A molecular mixture is a chemical mixture composed of two or more molecular entities with known concentration and stoichiometry.</p>
<h3 id="nucleicacidentity">NucleicAcidEntity</h3>
<p>A nucleic acid entity is a molecular entity characterized by availability in gene databases of nucleotide-based sequence representations of its precise sequence; for convenience of representation, partial sequences of various kinds are included.</p>
<h3 id="smallmolecule">SmallMolecule</h3>
<p>A small molecule entity is a molecular entity characterized by availability in small-molecule databases of SMILES, InChI, IUPAC, or other unambiguous representation of its precise chemical structure; for convenience of representation, any valid chemical representation is included, even if it is not strictly molecular (e.g., sodium ion).</p>
<h1 id="slots-in-subset">Slots in subset:</h1>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../actively_involved_in/">actively_involved_in</a></td>
<td>holds between a continuant and a process or function, where the continuant actively contributes to part or all of the process or function it realizes</td>
</tr>
<tr>
<td><a href="../actively_involves/">actively_involves</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../affects/">affects</a></td>
<td>Describes an entity that has an effect on the state or quality of another existing entity.</td>
</tr>
<tr>
<td><a href="../affects_likelihood_of/">affects_likelihood_of</a></td>
<td>Holds between two entities where the presence or application of one alters the chance that the other will come to be.</td>
</tr>
<tr>
<td><a href="../affects_sensitivity_to/">affects_sensitivity_to</a></td>
<td>holds between two chemical entities or genes or gene products where the action of one affects the susceptibility/sensitivity of a biological entity or system (e.g. an organism, cell, cellular component, macromolecular machine mixin, biological or pathological process) to the other.</td>
</tr>
<tr>
<td><a href="../anatomical_context_qualifier/">anatomical_context_qualifier</a></td>
<td>A statement qualifier representing an anatomical location where an relationship expressed in an association took place (can be a tissue, cell type, or sub-cellular location).</td>
</tr>
<tr>
<td><a href="../applied_to_treat/">applied_to_treat</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that the  substance, procedure, or activity was actually taken by one or more patients with the intent of treating the condition.</td>
</tr>
<tr>
<td><a href="../aspect_qualifier/">aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.</td>
</tr>
<tr>
<td><a href="../beneficial_in_models_for/">beneficial_in_models_for</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that the substance, procedure, or activity has been shown to be effective in alleviating, preventing, or delaying symptoms/ phenotypes associated with a disease, in a model system for that disease (e.g. a mouse, fly, cell line, etc).</td>
</tr>
<tr>
<td><a href="../biomarker_for/">biomarker_for</a></td>
<td>holds between a measurable chemical entity and a disease or phenotypic feature, where the entity is used as an indicator of the presence or state of the disease or feature.</td>
</tr>
<tr>
<td><a href="../broad_match/">broad_match</a></td>
<td>a list of terms from different schemas or terminology systems that have a broader, more general meaning. Broader terms are typically shown as parents in a hierarchy or tree.</td>
</tr>
<tr>
<td><a href="../capable_of/">capable_of</a></td>
<td>holds between a physical entity and process or function, where the continuant alone has the ability to carry out the process or function.</td>
</tr>
<tr>
<td><a href="../category/">category</a></td>
<td>Name of the high level ontology class in which this entity is categorized. Corresponds to the label for the biolink entity type class. In a neo4j database this MAY correspond to the neo4j label tag. In an RDF database it should be a biolink model class URI. This field is multi-valued. It should include values for ancestors of the biolink class; for example, a protein such as Shh would have category values <code>biolink:Protein</code>, <code>biolink:GeneProduct</code>, <code>biolink:MolecularEntity</code>. In an RDF database, nodes will typically have an rdf:type triples. This can be to the most specific biolink class, or potentially to a class more specific than something in biolink. For example, a sequence feature <code>f</code> may have a rdf:type assertion to a SO class such as TF_binding_site, which is more specific than anything in biolink. Here we would have categories {biolink:GenomicEntity, biolink:MolecularEntity, biolink:NamedThing}</td>
</tr>
<tr>
<td><a href="../causal_mechanism_qualifier/">causal_mechanism_qualifier</a></td>
<td>A statement qualifier representing a type of molecular control mechanism through which an effect of a chemical on a gene or gene product is mediated (e.g. 'agonism', 'inhibition', 'allosteric modulation', 'channel blocker')</td>
</tr>
<tr>
<td><a href="../caused_by/">caused_by</a></td>
<td>holds between two entities where the occurrence, existence, or activity of one is caused by the occurrence or generation of the other</td>
</tr>
<tr>
<td><a href="../causes/">causes</a></td>
<td>holds between two entities where the occurrence, existence, or activity of one causes the occurrence or generation of the other</td>
</tr>
<tr>
<td><a href="../chemically_similar_to/">chemically_similar_to</a></td>
<td>holds between one small molecule entity and another that it approximates for purposes of scientific study, in virtue of its exhibiting similar features of the studied entity.</td>
</tr>
<tr>
<td><a href="../close_match/">close_match</a></td>
<td>a list of terms from different schemas or terminology systems that have a semantically similar but not strictly equivalent, broader, or narrower meaning. Such terms often describe the same general concept from different ontological perspectives (e.g. drug as a type of chemical entity versus drug as a type of role borne by a chemical entity).</td>
</tr>
<tr>
<td><a href="../coexists_with/">coexists_with</a></td>
<td>holds between two entities that are co-located in the same aggregate object, process, or spatio-temporal region</td>
</tr>
<tr>
<td><a href="../colocalizes_with/">colocalizes_with</a></td>
<td>holds between two entities that are observed to be located in the same place.</td>
</tr>
<tr>
<td><a href="../composed_primarily_of/">composed_primarily_of</a></td>
<td>x composed_primarily_of_y if:more than half of the mass of x is made from parts of y.</td>
</tr>
<tr>
<td><a href="../condition_associated_with_gene/">condition_associated_with_gene</a></td>
<td>holds between a gene and a disease or phenotypic feature that may be influenced, contribute to, or be correlated with the gene or its alleles/products</td>
</tr>
<tr>
<td><a href="../condition_exacerbated_by/">condition_exacerbated_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../context_qualifier/">context_qualifier</a></td>
<td>Restricts the setting/context/location where the core concept (or qualified core concept) resides or occurs.</td>
</tr>
<tr>
<td><a href="../contributes_to/">contributes_to</a></td>
<td>holds between two entities where the occurrence, existence, or activity of one contributes to the occurrence or generation of the other</td>
</tr>
<tr>
<td><a href="../contribution_from/">contribution_from</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../correlated_with/">correlated_with</a></td>
<td>A relationship that holds between two concepts represented by variables for which a statistical correlation is believed to exist, as demonstrated using a correlation analysis method.</td>
</tr>
<tr>
<td><a href="../decreases_sensitivity_to/">decreases_sensitivity_to</a></td>
<td>holds between two chemical entities or genes or gene products where the action or effect of one decreases the susceptibility/sensitivity of a biological entity or system (e.g. an organism, cell, cellular component, macromolecular machine mixin, biological or pathological process) to the other</td>
</tr>
<tr>
<td><a href="../derivative_qualifier/">derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.</td>
</tr>
<tr>
<td><a href="../derives_from/">derives_from</a></td>
<td>holds between two distinct material entities, the new entity and the old entity, in which the new entity begins to exist when the old entity ceases to exist, and the new entity inherits the significant portion of the matter of the old entity</td>
</tr>
<tr>
<td><a href="../derives_into/">derives_into</a></td>
<td>holds between two distinct material entities, the old entity and the new entity, in which the new entity begins to exist when the old entity ceases to exist, and the new entity inherits the significant portion of the matter of the old entity</td>
</tr>
<tr>
<td><a href="../description/">description</a></td>
<td>a human-readable description of an entity</td>
</tr>
<tr>
<td><a href="../direction_qualifier/">direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree.</td>
</tr>
<tr>
<td><a href="../disease_context_qualifier/">disease_context_qualifier</a></td>
<td>A context qualifier representing a disease or condition in which a relationship expressed in an association took place.</td>
</tr>
<tr>
<td><a href="../disrupts/">disrupts</a></td>
<td>describes a relationship where one entity degrades or interferes with the structure, function, or occurrence of another.</td>
</tr>
<tr>
<td><a href="../enabled_by/">enabled_by</a></td>
<td>holds between a process and a physical entity, where the physical entity executes the process</td>
</tr>
<tr>
<td><a href="../enables/">enables</a></td>
<td>holds between a physical entity and a process, where the physical entity executes the process</td>
</tr>
<tr>
<td><a href="../exact_match/">exact_match</a></td>
<td>holds between two entities that have strictly equivalent meanings, with a high degree of confidence</td>
</tr>
<tr>
<td><a href="../expressed_in/">expressed_in</a></td>
<td>holds between a gene or gene product and an anatomical entity in which it is expressed</td>
</tr>
<tr>
<td><a href="../expresses/">expresses</a></td>
<td>holds between an anatomical entity and gene or gene product that is expressed there</td>
</tr>
<tr>
<td><a href="../food_component_of/">food_component_of</a></td>
<td>holds between a one or more chemical entities present in food, irrespective of nutritional value (i.e. could also be a contaminant or additive)</td>
</tr>
<tr>
<td><a href="../form_or_variant_qualifier/">form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.</td>
</tr>
<tr>
<td><a href="../frequency_qualifier/">frequency_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state how frequent the phenotype is observed in the subject</td>
</tr>
<tr>
<td><a href="../gene_associated_with_condition/">gene_associated_with_condition</a></td>
<td>holds between a gene and a disease or phenotypic feature that the gene or its alleles/products may influence, contribute to, or correlate with</td>
</tr>
<tr>
<td><a href="../gene_product_of/">gene_product_of</a></td>
<td>definition x has gene product of y if and only if y is a gene (SO:0000704) that participates in some gene expression process (GO:0010467) where the output of thatf process is either y or something that is ribosomally translated from x</td>
</tr>
<tr>
<td><a href="../gene_fusion_with/">gene_fusion_with</a></td>
<td>holds between two independent genes that have fused through translocation, interstitial deletion, or chromosomal inversion to form a new, hybrid gene. Fusion genes are often implicated in various neoplasms and cancers.</td>
</tr>
<tr>
<td><a href="../genetic_neighborhood_of/">genetic_neighborhood_of</a></td>
<td>holds between two genes located nearby one another on a chromosome.</td>
</tr>
<tr>
<td><a href="../genetically_associated_with/">genetically_associated_with</a></td>
<td></td>
</tr>
<tr>
<td><a href="../genetically_interacts_with/">genetically_interacts_with</a></td>
<td>holds between two genes whose phenotypic effects are dependent on each other in some way - such that their combined phenotypic effects are the result of some interaction between the activity of their gene products. Examples include epistasis and synthetic lethality.</td>
</tr>
<tr>
<td><a href="../has_active_ingredient/">has_active_ingredient</a></td>
<td>holds between a drug and a molecular entity in which the latter is a part of the former, and is a biologically active component</td>
</tr>
<tr>
<td><a href="../has_biomarker/">has_biomarker</a></td>
<td>holds between a disease or phenotypic feature and a measurable chemical entity that is used as an indicator of the presence or state of the disease or feature.</td>
</tr>
<tr>
<td># metabolite</td>
<td></td>
</tr>
<tr>
<td><a href="../has_excipient/">has_excipient</a></td>
<td>holds between a drug and a molecular entities in which the latter is a part of the former, and is a biologically inactive component</td>
</tr>
<tr>
<td><a href="../has_food_component/">has_food_component</a></td>
<td>holds between food and one or more chemical entities composing it, irrespective of nutritional value (i.e. could also be a contaminant or additive)</td>
</tr>
<tr>
<td><a href="../has_gene_product/">has_gene_product</a></td>
<td>holds between a gene and a transcribed and/or translated product generated from it</td>
</tr>
<tr>
<td><a href="../has_input/">has_input</a></td>
<td>holds between a process and a continuant, where the continuant is an input into the process</td>
</tr>
<tr>
<td><a href="../has_member/">has_member</a></td>
<td>Defines a mereological relation between a collection and an item.</td>
</tr>
<tr>
<td><a href="../has_metabolite/">has_metabolite</a></td>
<td>holds between two molecular entities in which the second one is derived from the first one as a product of metabolism</td>
</tr>
<tr>
<td><a href="../has_mode_of_inheritance/">has_mode_of_inheritance</a></td>
<td>Relates a disease or phenotypic feature to its observed genetic segregation and assumed associated underlying DNA manifestation (i.e. autosomal, sex or mitochondrial chromosome).</td>
</tr>
<tr>
<td><a href="../has_nutrient/">has_nutrient</a></td>
<td>one or more nutrients which are growth factors for a living organism</td>
</tr>
<tr>
<td><a href="../has_output/">has_output</a></td>
<td>holds between a process and a continuant, where the continuant is an output of the process</td>
</tr>
<tr>
<td><a href="../has_part/">has_part</a></td>
<td>holds between wholes and their parts (material entities or processes)</td>
</tr>
<tr>
<td><a href="../has_participant/">has_participant</a></td>
<td>holds between a process and a continuant, where the continuant is somehow involved in the process</td>
</tr>
<tr>
<td><a href="../has_phenotype/">has_phenotype</a></td>
<td>holds between a biological entity and a phenotype, where a phenotype is construed broadly as any kind of quality of an organism part, a collection of these qualities, or a change in quality or qualities (e.g. abnormally increased temperature). In SNOMEDCT, disorders with keyword 'characterized by' should translate into this predicate.</td>
</tr>
<tr>
<td><a href="../has_plasma_membrane_part/">has_plasma_membrane_part</a></td>
<td>Holds between a cell c and a protein complex or protein p if and only if that cell has as part a plasma_membrane[GO:0005886], and that plasma membrane has p as part.</td>
</tr>
<tr>
<td><a href="../homologous_to/">homologous_to</a></td>
<td>holds between two biological entities that have common evolutionary origin</td>
</tr>
<tr>
<td><a href="../id/">id</a></td>
<td>A unique identifier for an entity. Must be either a CURIE shorthand for a URI or a complete URI</td>
</tr>
<tr>
<td><a href="../in_cell_population_with/">in_cell_population_with</a></td>
<td>holds between two genes or gene products that are expressed in the same cell type or population</td>
</tr>
<tr>
<td><a href="../in_clinical_trials_for/">in_clinical_trials_for</a></td>
<td>Holds between an intervention and a medical condition, and reports that a clinical trial  is being or has been performed in human patients to test the potential of the intervention to treat the medical condition (e.g. to ameliorate, stabilize, or cure the condition, or to delay, prevent, or reduce the risk of it manifesting in the first place).</td>
</tr>
<tr>
<td><a href="../in_complex_with/">in_complex_with</a></td>
<td>holds between two genes or gene products that are part of (or code for products that are part of) in the same macromolecular complex</td>
</tr>
<tr>
<td><a href="../in_pathway_with/">in_pathway_with</a></td>
<td>holds between two genes or gene products that are part of in the same biological pathway</td>
</tr>
<tr>
<td><a href="../in_preclinical_trials_for/">in_preclinical_trials_for</a></td>
<td>Holds between an  substance, procedure, or activity and a medical condition, and reports that a pre-clinical study has been performed specifically to test the potential of the  substance, procedure, or activity to treat the medical condition  (i.e. to ameliorate, stabilize, or cure the condition, or to delay, prevent, or reduce the risk of it manifesting in the first place).</td>
</tr>
<tr>
<td><a href="../in_taxon/">in_taxon</a></td>
<td>connects an entity to its taxonomic classification. Only certain kinds of entities can be taxonomically classified; see 'thing with taxon'</td>
</tr>
<tr>
<td><a href="../in_taxon_label/">in_taxon_label</a></td>
<td>The human readable scientific name for the taxon of the entity.</td>
</tr>
<tr>
<td><a href="../increases_sensitivity_to/">increases_sensitivity_to</a></td>
<td>holds between two chemical entities or genes or gene products where the action or effect of one increases the susceptibility/sensitivity of a biological entity or system (e.g. an organism, cell, cellular component, macromolecular machine mixin, biological or pathological process) to the other</td>
</tr>
<tr>
<td><a href="../interacts_with/">interacts_with</a></td>
<td>holds between any two entities that directly or indirectly interact with each other</td>
</tr>
<tr>
<td><a href="../iri/">iri</a></td>
<td>An IRI for an entity. This is determined by the id using expansion rules.</td>
</tr>
<tr>
<td><a href="../is_active_ingredient_of/">is_active_ingredient_of</a></td>
<td>holds between a molecular entity and a drug, in which the former is a part of the latter, and is a biologically active component</td>
</tr>
<tr>
<td><a href="../is_excipient_of/">is_excipient_of</a></td>
<td>holds between a molecular entity and a drug in which the former is a part of the latter, and is a biologically inactive component</td>
</tr>
<tr>
<td><a href="../is_input_of/">is_input_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../is_metabolite_of/">is_metabolite_of</a></td>
<td>holds between two molecular entities in which the first one is derived from the second one as a product of metabolism</td>
</tr>
<tr>
<td><a href="../is_output_of/">is_output_of</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../located_in/">located_in</a></td>
<td>holds between a material entity and a material entity or site within which it is located (but of which it is not considered a part)</td>
</tr>
<tr>
<td><a href="../location_of/">location_of</a></td>
<td>holds between material entity or site and a material entity that is located within it (but not considered a part of it)</td>
</tr>
<tr>
<td><a href="../manifestation_of/">manifestation_of</a></td>
<td>that part of a phenomenon which is directly observable or visibly expressed, or which gives evidence to the underlying process; used in SemMedDB for linking things like dysfunctions and processes to some disease or syndrome</td>
</tr>
<tr>
<td><a href="../member_of/">member_of</a></td>
<td>Defines a mereological relation between a item and a collection.</td>
</tr>
<tr>
<td><a href="../model_of/">model_of</a></td>
<td>holds between a thing and some other thing it approximates for purposes of scientific study, in virtue of its exhibiting similar features of the studied entity.</td>
</tr>
<tr>
<td><a href="../name/">name</a></td>
<td>A human-readable name for an attribute or entity.</td>
</tr>
<tr>
<td><a href="../narrow_match/">narrow_match</a></td>
<td>a list of terms from different schemas or terminology systems that have a narrower, more specific meaning. Narrower terms are typically shown as children in a hierarchy or tree.</td>
</tr>
<tr>
<td><a href="../negatively_correlated_with/">negatively_correlated_with</a></td>
<td>A relationship that holds between two concepts represented by variables for which a statistical correlation is demonstrated, wherein variable values move in opposite directions (i.e. increased in one or presence of one correlates with a decrease or absence of the other).</td>
</tr>
<tr>
<td><a href="../nutrient_of/">nutrient_of</a></td>
<td>holds between a one or more chemical entities present in food, irrespective of nutritional value (i.e. could also be a contaminant or additive)</td>
</tr>
<tr>
<td><a href="../object_aspect_qualifier/">object_aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.  This qualifier specifies a change in the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="../object_context_qualifier/">object_context_qualifier</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../object_derivative_qualifier/">object_derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.  This qualifier is for the object of an association (or statement).</td>
</tr>
<tr>
<td><a href="../object_direction_qualifier/">object_direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree. This qualifier qualifies the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="../object_form_or_variant_qualifier/">object_form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.  This qualifier specifies a change in the object of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="../object_part_qualifier/">object_part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).  This qualifier is for the object of an association (or statement).</td>
</tr>
<tr>
<td><a href="../object_specialization_qualifier/">object_specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="../occurs_in/">occurs_in</a></td>
<td>holds between a process and a material entity or site within which the process occurs</td>
</tr>
<tr>
<td><a href="../occurs_together_in_literature_with/">occurs_together_in_literature_with</a></td>
<td>holds between two entities where their co-occurrence is correlated by counts of publications in which both occur, using some threshold of occurrence as defined by the edge provider.</td>
</tr>
<tr>
<td><a href="../onset_qualifier/">onset_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state when the phenotype appears is in the subject.</td>
</tr>
<tr>
<td><a href="../orthologous_to/">orthologous_to</a></td>
<td>a homology relationship between entities (typically genes) that diverged after a speciation event.</td>
</tr>
<tr>
<td><a href="../overlaps/">overlaps</a></td>
<td>holds between entities that overlap in their extents (materials or processes)</td>
</tr>
<tr>
<td><a href="../paralogous_to/">paralogous_to</a></td>
<td>a homology relationship that holds between entities (typically genes) that diverged after a duplication event.</td>
</tr>
<tr>
<td><a href="../part_of/">part_of</a></td>
<td>holds between parts and wholes (material entities or processes)</td>
</tr>
<tr>
<td><a href="../part_qualifier/">part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).</td>
</tr>
<tr>
<td><a href="../participates_in/">participates_in</a></td>
<td>holds between a continuant and a process, where the continuant is somehow involved in the process</td>
</tr>
<tr>
<td><a href="../physically_interacts_with/">physically_interacts_with</a></td>
<td>holds between two entities that make physical contact as part of some interaction. does not imply a causal relationship.</td>
</tr>
<tr>
<td><a href="../positively_correlated_with/">positively_correlated_with</a></td>
<td>A relationship that holds between two concepts represented by variables for which a statistical correlation is demonstrated, wherein variable values move together in the same direction (i.e. increased in one or presence of one correlates with an increase or presence of the other).</td>
</tr>
<tr>
<td><a href="../preceded_by/">preceded_by</a></td>
<td>holds between two processes, where the other is completed before the one begins</td>
</tr>
<tr>
<td><a href="../precedes/">precedes</a></td>
<td>holds between two processes, where one completes before the other begins</td>
</tr>
<tr>
<td><a href="../predisposes_to_condition/">predisposes_to_condition</a></td>
<td>Holds between two entities where the presence or application of one increases the chance that the other will come to be.</td>
</tr>
<tr>
<td><a href="../preventative_for_condition/">preventative_for_condition</a></td>
<td>Holds between a substance, procedure, or activity and a medical condition (disease or phenotypic feature), and states that the  substance, procedure, or activity is able to prevent it manifesting in the first place.</td>
</tr>
<tr>
<td><a href="../produces/">produces</a></td>
<td>holds between a material entity and a product that is generated through the intentional actions or functioning of the material entity</td>
</tr>
<tr>
<td><a href="../promotes_condition/">promotes_condition</a></td>
<td>Holds between a substance, procedure, or activity and a medical condition (disease or phenotypic feature), and states that the  substance, procedure, or activity is able to promote it manifesting in the first place.</td>
</tr>
<tr>
<td><a href="../qualifier/">qualifier</a></td>
<td>grouping slot for all qualifiers on an edge.  useful for testing compliance with association classes</td>
</tr>
<tr>
<td><a href="../resource_id/">resource_id</a></td>
<td>The CURIE for an Information Resource that served as a source of knowledge expressed in an Edge, or a source of data used to generate this knowledge.</td>
</tr>
<tr>
<td><a href="../resource_role/">resource_role</a></td>
<td>The role played by the InformationResource in serving as a source for an Edge. Note that a given Edge should have one and only one 'primary' source, and may have any number of 'aggregator' or 'supporting data' sources.</td>
</tr>
<tr>
<td><a href="../retrieval_source_ids/">retrieval_source_ids</a></td>
<td>A list of retrieval sources that served as a source of knowledge expressed in an Edge, or a source of data used to generate this knowledge.</td>
</tr>
<tr>
<td><a href="../same_as/">same_as</a></td>
<td>holds between two entities that are considered equivalent to each other</td>
</tr>
<tr>
<td><a href="../semmed_agreement_count/">semmed_agreement_count</a></td>
<td>The number of times this concept has been asserted in the SemMedDB literature database.</td>
</tr>
<tr>
<td><a href="../sensitivity_affected_by/">sensitivity_affected_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../sensitivity_decreased_by/">sensitivity_decreased_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../sensitivity_increased_by/">sensitivity_increased_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../severity_qualifier/">severity_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state how severe the phenotype is in the subject</td>
</tr>
<tr>
<td><a href="../sex_qualifier/">sex_qualifier</a></td>
<td>a qualifier used in a phenotypic association to state whether the association is specific to a particular sex.</td>
</tr>
<tr>
<td><a href="../similar_to/">similar_to</a></td>
<td>holds between an entity and some other entity with similar features.</td>
</tr>
<tr>
<td><a href="../species_context_qualifier/">species_context_qualifier</a></td>
<td>A statement qualifier representing a taxonomic category of species in which a relationship expressed in an association took place.</td>
</tr>
<tr>
<td><a href="../stage_qualifier/">stage_qualifier</a></td>
<td>stage during which gene or protein expression of takes place.</td>
</tr>
<tr>
<td><a href="../statement_qualifier/">statement_qualifier</a></td>
<td>A property that qualifies the entirety of the statement made in an association.  It applies to both a fully qualified subject and a fully qualified object as well as the predicate and qualified predicate in an association.</td>
</tr>
<tr>
<td><a href="../subclass_of/">subclass_of</a></td>
<td>holds between two classes where the domain class is a specialization of the range class</td>
</tr>
<tr>
<td><a href="../subject_aspect_qualifier/">subject_aspect_qualifier</a></td>
<td>Composes with the core concept to describe new concepts of a different ontological type. e.g. a process in which the core concept participates, a function/activity/role held by the core concept, or a characteristic/quality that inheres in the core concept.  The purpose of the aspect slot is to indicate what aspect is being affected in an 'affects' association.  This qualifier specifies a change in the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="../subject_context_qualifier/">subject_context_qualifier</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../subject_derivative_qualifier/">subject_derivative_qualifier</a></td>
<td>A qualifier that composes with a core subject/object  concept to describe something that is derived from the core concept.  For example, the qualifier ‘metabolite’ combines with a ‘Chemical X’ core concept to express the composed concept ‘a metabolite of Chemical X’.  This qualifier is for the subject of an association (or statement).</td>
</tr>
<tr>
<td><a href="../subject_direction_qualifier/">subject_direction_qualifier</a></td>
<td>Composes with the core concept (+ aspect if provided) to describe a change in its direction or degree. This qualifier qualifies the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="../subject_form_or_variant_qualifier/">subject_form_or_variant_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a specific type, variant, alternative version of this concept. The composed concept remains a subtype or instance of the core concept. For example, the qualifier ‘mutation’ combines with the core concept ‘Gene X’ to express the compose concept ‘a mutation of Gene X’.  This qualifier specifies a change in the subject of an association (aka: statement).</td>
</tr>
<tr>
<td><a href="../subject_part_qualifier/">subject_part_qualifier</a></td>
<td>defines a specific part/component of the core concept (used in cases there this specific part has no IRI we can use to directly represent it, e.g. 'ESR1 transcript' q: polyA tail).  This qualifier is for the subject of an association (or statement).</td>
</tr>
<tr>
<td><a href="../subject_specialization_qualifier/">subject_specialization_qualifier</a></td>
<td>A qualifier that composes with a core subject/object concept to define a more specific version of the subject concept, specifically using an ontology term that is not a subclass or descendant of the core concept and in the vast majority of cases, is of a different ontological namespace than the category or namespace of the subject identifier.</td>
</tr>
<tr>
<td><a href="../superclass_of/">superclass_of</a></td>
<td>holds between two classes where the domain class is a super class of the range class</td>
</tr>
<tr>
<td><a href="../support_graphs/">support_graphs</a></td>
<td>A list of knowledge graphs that support the existence of this association.</td>
</tr>
<tr>
<td><a href="../synonym/">synonym</a></td>
<td>Alternate human-readable names for a thing</td>
</tr>
<tr>
<td><a href="../taxon/">taxon</a></td>
<td>A property that indicates the taxonomic classification of an entity. Values for this slot should be from the NCBITaxon ontology.</td>
</tr>
<tr>
<td><a href="../treated_by/">treated_by</a></td>
<td>None</td>
</tr>
<tr>
<td><a href="../treats/">treats</a></td>
<td>Holds between an intervention (substance, procedure, or activity) and a medical condition (disease or phenotypic feature), and states that the intervention is, in some population(s), able to ameliorate, stabilize, or cure the condition or delay, prevent, or reduce the risk of it manifesting in the first place. ‘Treats’ edges should be asserted (knowledge_level: assertion) only in cases where there is strong supporting evidence - i.e. in some population(s) the intervention is approved for the condition, passed phase 3 or in phase 4 trials for the condition, or is an otherwise established treatment in the medical community (e.g. a widely-accepted or formally recommended off-label use). In the absence of such evidence, weaker predicates should be used in asserted edges (e.g. ‘in clinical trials for’ or ‘beneficial in models of’). ‘Treats’ edges based on weaker or indirect forms of evidence can however be created as predictions (knowledge_level: prediction) and should point to the more foundational asserted edges that support them.</td>
</tr>
<tr>
<td><a href="../treats_or_applied_or_studied_to_treat/">treats_or_applied_or_studied_to_treat</a></td>
<td>Holds between an substance, procedure, or activity and a medical condition (disease or phenotypic feature), and states that the substance, procedure, or activity is able to treat the condition, has been observed to be taken/prescribed in practice with the intent of treating the condition, or has been interrogated in a scientific study that hypothesized an ability to treat the condition (in humans or other biological systems/organisms).</td>
</tr>
<tr>
<td><a href="../xenologous_to/">xenologous_to</a></td>
<td>a homology relationship characterized by an interspecies (horizontal) transfer since the common ancestor.</td>
</tr>
<tr>
<td><a href="../xref/">xref</a></td>
<td>A database cross reference or alternative identifier for a NamedThing or edge between two NamedThings.  This property should point to a database record or webpage that supports the existence of the edge, or gives more detail about the edge. This property can be used on a node or edge to provide multiple URIs or CURIE cross references.</td>
</tr>
</tbody>
</table>
<h1 id="enumerations-in-subset">Enumerations in subset:</h1>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p>| <a href="../AgentTypeEnum/">AgentTypeEnum</a> | None |</p>
<p>| <a href="../FDAIDAAdverseEventEnum/">FDAIDAAdverseEventEnum</a> | please consult with the FDA guidelines as proposed in this document: https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfcfr/cfrsearch.cfm?fr=312.32 |</p>
<p>| <a href="../KnowledgeLevelEnum/">KnowledgeLevelEnum</a> | None |</p>
<p>| <a href="../ResourceRoleEnum/">ResourceRoleEnum</a> | The role played by the information reource in serving as a source for an edge in a TRAPI message. Note that a given Edge should have one and only one 'primary' source, and may have any number of 'aggregator' or 'supporting data' sources.  This enumeration is found in Biolink Model, but is repeated here for convenience. |</p>









  




                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.d50fe291.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../assets/javascripts/bundle.56ea9cef.min.js"></script>
      
    
  </body>
</html>